From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Node.js GitHub Bot" <github-bot@iojs.org>
Date: Sun, 24 Aug 2025 00:12:43 +0100
Subject: [PATCH 42/74] crypto: update root certificates to NSS 3.114
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This is the certdata.txt[0] from NSS 3.114.

This is the version of NSS that shipped in Firefox 142.0 on 2025-08-19.

Certificates added:
- TrustAsia TLS ECC Root CA
- TrustAsia TLS RSA Root CA
- SwissSign RSA TLS Root CA 2022 - 1

Certificates removed:
- GlobalSign Root CA
- Entrust.net Premium 2048 Secure Server CA
- Baltimore CyberTrust Root
- Comodo AAA Services root
- XRamp Global CA Root
- Go Daddy Class 2 CA
- Starfield Class 2 CA

[0] https://raw.githubusercontent.com/nss-dev/nss/refs/tags/NSS_3_114_RTM/lib/ckfw/builtins/certdata.txt

PR-URL: https://github.com/nodejs/node/pull/59571
Reviewed-By: Richard Lau <richard.lau@ibm.com>
Reviewed-By: Michaël Zasso <targos@protonmail.com>
Reviewed-By: Luigi Pinca <luigipinca@gmail.com>
(cherry picked from commit 6277910a15a4272ffe4b08e9fae9b6618530ff2e)
---
 src/node_root_certs.h |  239 +++------
 tools/certdata.txt    | 1066 +++++++++++++++++++++++++++++++++++------
 2 files changed, 1000 insertions(+), 305 deletions(-)

diff --git a/src/node_root_certs.h b/src/node_root_certs.h
index 69634bca1d..9bf95d49f4 100644
--- a/src/node_root_certs.h
+++ b/src/node_root_certs.h
@@ -1,71 +1,5 @@
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-/* GlobalSign Root CA */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIDdTCCAl2gAwIBAgILBAAAAAABFUtaw5QwDQYJKoZIhvcNAQEFBQAwVzELMAkGA1UEBhMC\n"
-"QkUxGTAXBgNVBAoTEEdsb2JhbFNpZ24gbnYtc2ExEDAOBgNVBAsTB1Jvb3QgQ0ExGzAZBgNV\n"
-"BAMTEkdsb2JhbFNpZ24gUm9vdCBDQTAeFw05ODA5MDExMjAwMDBaFw0yODAxMjgxMjAwMDBa\n"
-"MFcxCzAJBgNVBAYTAkJFMRkwFwYDVQQKExBHbG9iYWxTaWduIG52LXNhMRAwDgYDVQQLEwdS\n"
-"b290IENBMRswGQYDVQQDExJHbG9iYWxTaWduIFJvb3QgQ0EwggEiMA0GCSqGSIb3DQEBAQUA\n"
-"A4IBDwAwggEKAoIBAQDaDuaZjc6j40+Kfvvxi4Mla+pIH/EqsLmVEQS98GPR4mdmzxzdzxtI\n"
-"K+6NiY6arymAZavpxy0Sy6scTHAHoT0KMM0VjU/43dSMUBUc71DuxC73/OlS8pF94G3VNTCO\n"
-"XkNz8kHp1Wrjsok6Vjk4bwY8iGlbKk3Fp1S4bInMm/k8yuX9ifUSPJJ4ltbcdG6TRGHRjcdG\n"
-"snUOhugZitVtbNV4FpWi6cgKOOvyJBNPc1STE4U6G7weNLWLBYy5d4ux2x8gkasJU26Qzns3\n"
-"dLlwR5EiUWMWea6xrkEmCMgZK9FGqkjWZCrXgzT/LCrBbBlDSgeF59N89iFo7+ryUp9/k5DP\n"
-"AgMBAAGjQjBAMA4GA1UdDwEB/wQEAwIBBjAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBRg\n"
-"e2YaRQ2XyolQL30EzTSo//z9SzANBgkqhkiG9w0BAQUFAAOCAQEA1nPnfE920I2/7LqivjTF\n"
-"KDK1fPxsnCwrvQmeU79rXqoRSLblCKOzyj1hTdNGCbM+w6DjY1Ub8rrvrTnhQ7k4o+YviiY7\n"
-"76BQVvnGCv04zcQLcFGUl5gE38NflNUVyRRBnMRddWQVDf9VMOyGj/8N7yy5Y0b2qvzfvGn9\n"
-"LhJIZJrglfCm7ymPAbEVtQwdpf5pLGkkeB6zpxxxYu7KyJesF12KwvhHhm4qxFYxldBniYUr\n"
-"+WymXUadDKqC5JlR3XC321Y9YeRq4VzW9v493kHMB65jUr9TU/Qr6cf9tveCX4XSQRjbgbME\n"
-"HMUfpIBvFSDJ3gyICh3WZlXi/EjJKSZp4A==\n"
-"-----END CERTIFICATE-----",
-
-/* Entrust.net Premium 2048 Secure Server CA */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIEKjCCAxKgAwIBAgIEOGPe+DANBgkqhkiG9w0BAQUFADCBtDEUMBIGA1UEChMLRW50cnVz\n"
-"dC5uZXQxQDA+BgNVBAsUN3d3dy5lbnRydXN0Lm5ldC9DUFNfMjA0OCBpbmNvcnAuIGJ5IHJl\n"
-"Zi4gKGxpbWl0cyBsaWFiLikxJTAjBgNVBAsTHChjKSAxOTk5IEVudHJ1c3QubmV0IExpbWl0\n"
-"ZWQxMzAxBgNVBAMTKkVudHJ1c3QubmV0IENlcnRpZmljYXRpb24gQXV0aG9yaXR5ICgyMDQ4\n"
-"KTAeFw05OTEyMjQxNzUwNTFaFw0yOTA3MjQxNDE1MTJaMIG0MRQwEgYDVQQKEwtFbnRydXN0\n"
-"Lm5ldDFAMD4GA1UECxQ3d3d3LmVudHJ1c3QubmV0L0NQU18yMDQ4IGluY29ycC4gYnkgcmVm\n"
-"LiAobGltaXRzIGxpYWIuKTElMCMGA1UECxMcKGMpIDE5OTkgRW50cnVzdC5uZXQgTGltaXRl\n"
-"ZDEzMDEGA1UEAxMqRW50cnVzdC5uZXQgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkgKDIwNDgp\n"
-"MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArU1LqRKGsuqjIAcVFmQqK0vRvwtK\n"
-"TY7tgHalZ7d4QMBzQshowNtTK91euHaYNZOLGp18EzoOH1u3Hs/lJBQesYGpjX24zGtLA/EC\n"
-"DNyrpUAkAH90lKGdCCmziAv1h3edVc3kw37XamSrhRSGlVuXMlBvPci6Zgzj/L24ScF2iUkZ\n"
-"/cCovYmjZy/Gn7xxGWC4LeksyZB2ZnuU4q941mVTXTzWnLLPKQP5L6RQstRIzgUyVYr9smRM\n"
-"DuSYB3Xbf9+5CFVghTAp+XtIpGmG4zU/HoZdenoVve8AjhUiVBcAkCaTvA5JaJG/+EfTnZVC\n"
-"wQ5N328mz8MYIWJmQ3DW1cAH4QIDAQABo0IwQDAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/\n"
-"BAUwAwEB/zAdBgNVHQ4EFgQUVeSB0RGAvtiJuQijMfmhJAkWuXAwDQYJKoZIhvcNAQEFBQAD\n"
-"ggEBADubj1abMOdTmXx6eadNl9cZlZD7Bh/KM3xGY4+WZiT6QBshJ8rmcnPyT/4xmf3IDExo\n"
-"U8aAghOY+rat2l098c5u9hURlIIM7j+VrxGrD9cv3h8Dj1csHsm7mhpElesYT6YfzX1XEC+b\n"
-"BAlahLVu2B064dae0Wx5XnkcFMXj0EyTO2U87d89vqbllRrDtRnDvV5bu/8j72gZyxKTJ1wD\n"
-"LW8w0B62GqzeWvfRqqgnpv55gcR5mTNXuhKwqeBCbJPKVt7+bYQLCIt+jerXmCHG8+c8eS9e\n"
-"nNFMFY3h7CI3zJpDC5fcgJCNs2ebb0gIFVbPv/ErfF6adulZkMV8gzURZVE=\n"
-"-----END CERTIFICATE-----",
-
-/* Baltimore CyberTrust Root */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIDdzCCAl+gAwIBAgIEAgAAuTANBgkqhkiG9w0BAQUFADBaMQswCQYDVQQGEwJJRTESMBAG\n"
-"A1UEChMJQmFsdGltb3JlMRMwEQYDVQQLEwpDeWJlclRydXN0MSIwIAYDVQQDExlCYWx0aW1v\n"
-"cmUgQ3liZXJUcnVzdCBSb290MB4XDTAwMDUxMjE4NDYwMFoXDTI1MDUxMjIzNTkwMFowWjEL\n"
-"MAkGA1UEBhMCSUUxEjAQBgNVBAoTCUJhbHRpbW9yZTETMBEGA1UECxMKQ3liZXJUcnVzdDEi\n"
-"MCAGA1UEAxMZQmFsdGltb3JlIEN5YmVyVHJ1c3QgUm9vdDCCASIwDQYJKoZIhvcNAQEBBQAD\n"
-"ggEPADCCAQoCggEBAKMEuyKrmD1X6CZymrV51Cni4eiVgLGw41uOKymaZN+hXe2wCQVt2ygu\n"
-"zmKiYv60iNoS6zjrIZ3AQSsBUnuId9Mcj8e6uYi1agnnc+gRQKfRzMpijS3ljwumUNKoUMMo\n"
-"6vWrJYeKmpYcqWe4PwzV9/lSEy/CG9VwcPCPwBLKBsua4dnKM3p31vjsufFoREJIE9LAwqSu\n"
-"XmD+tqYF/LTdB1kC1FkYmGP1pWPgkAx9XbIGevOF6uvUA65ehD5f/xXtabz5OTZydc93Uk3z\n"
-"yZAsuT3lySNTPx8kmCFcB5kpvcY67Oduhjprl3RjM71oGDHweI12v/yejl0qhqdNkNwnGjkC\n"
-"AwEAAaNFMEMwHQYDVR0OBBYEFOWdWTCCR1jMrPoIVDaGezq1BE3wMBIGA1UdEwEB/wQIMAYB\n"
-"Af8CAQMwDgYDVR0PAQH/BAQDAgEGMA0GCSqGSIb3DQEBBQUAA4IBAQCFDF2O5G9RaEIFoN27\n"
-"TyclhAO992T9Ldcw46QQF+vaKSm2eT929hkTI7gQCvlYpNRhcL0EYWoSihfVCr3FvDB81ukM\n"
-"JY2GQE/szKN+OMY3EU/t3WgxjkzSswF07r51XgdIGn9w/xZchMB5hbgF/X++ZRGjD8ACtPhS\n"
-"NzkE1akxehi/oCr0Epn3o0WC4zxe9Z2etciefC7IpJ5OCBRLbf1wbWsaY71k5h+3zvDyny67\n"
-"G7fyUIhzksLi4xaNmjICq44Y3ekQEe5+NauQrz4wlHrQMz2nZQ/1/I6eYs9HRCwBXbsdtTLS\n"
-"R9I4LtD+gdwyah617jzV/OeBHRnDJELqYzmp\n"
-"-----END CERTIFICATE-----",
-
 /* Entrust Root Certification Authority */
 "-----BEGIN CERTIFICATE-----\n"
 "MIIEkTCCA3mgAwIBAgIERWtQVDANBgkqhkiG9w0BAQUFADCBsDELMAkGA1UEBhMCVVMxFjAU\n"
@@ -92,30 +26,6 @@
 "j2A781q0tHuu2guQOHXvgR1m0vdXcDazv/wor3ElhVsT/h5/WrQ8\n"
 "-----END CERTIFICATE-----",
 
-/* Comodo AAA Services root */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIEMjCCAxqgAwIBAgIBATANBgkqhkiG9w0BAQUFADB7MQswCQYDVQQGEwJHQjEbMBkGA1UE\n"
-"CAwSR3JlYXRlciBNYW5jaGVzdGVyMRAwDgYDVQQHDAdTYWxmb3JkMRowGAYDVQQKDBFDb21v\n"
-"ZG8gQ0EgTGltaXRlZDEhMB8GA1UEAwwYQUFBIENlcnRpZmljYXRlIFNlcnZpY2VzMB4XDTA0\n"
-"MDEwMTAwMDAwMFoXDTI4MTIzMTIzNTk1OVowezELMAkGA1UEBhMCR0IxGzAZBgNVBAgMEkdy\n"
-"ZWF0ZXIgTWFuY2hlc3RlcjEQMA4GA1UEBwwHU2FsZm9yZDEaMBgGA1UECgwRQ29tb2RvIENB\n"
-"IExpbWl0ZWQxITAfBgNVBAMMGEFBQSBDZXJ0aWZpY2F0ZSBTZXJ2aWNlczCCASIwDQYJKoZI\n"
-"hvcNAQEBBQADggEPADCCAQoCggEBAL5AnfRu4ep2hxxNRUSOvkbIgwadwSr+GB+O5AL686td\n"
-"UIoWMQuaBtDFcCLNSS1UY8y2bmhGC1Pqy0wkwLxyTurxFa70VJoSCsN6sjNg4tqJVfMiWPPe\n"
-"3M/vg4aijJRPn2jymJBGhCfHdr/jzDUsi14HZGWCwEiwqJH5YZ92IFCokcdmtet4YgNW8Ioa\n"
-"E+oxox6gmf049vYnMlhvB/VruPsUK6+3qszWY19zjNoFmag4qMsXeDZRrOme9Hg6jc8P2ULi\n"
-"mAyrL58OAd7vn5lJ8S3frHRNG5i1R8XlKdH5kBjHYpy+g8cmez6KJcfA3Z3mNWgQIJ2P2N7S\n"
-"w4ScDV7oL8kCAwEAAaOBwDCBvTAdBgNVHQ4EFgQUoBEKIz6W8Qfs4q8p74Klf9AwpLQwDgYD\n"
-"VR0PAQH/BAQDAgEGMA8GA1UdEwEB/wQFMAMBAf8wewYDVR0fBHQwcjA4oDagNIYyaHR0cDov\n"
-"L2NybC5jb21vZG9jYS5jb20vQUFBQ2VydGlmaWNhdGVTZXJ2aWNlcy5jcmwwNqA0oDKGMGh0\n"
-"dHA6Ly9jcmwuY29tb2RvLm5ldC9BQUFDZXJ0aWZpY2F0ZVNlcnZpY2VzLmNybDANBgkqhkiG\n"
-"9w0BAQUFAAOCAQEACFb8AvCb6P+k+tZ7xkSAzk/ExfYAWMymtrwUSWgEdujm7l3sAg9g1o1Q\n"
-"GE8mTgHj5rCl7r+8dFRBv/38ErjHT1r0iWAFf2C3BUrz9vHCv8S5dIa2LX1rzNLzRt0vxuBq\n"
-"w8M0Ayx9lt1awg6nCpnBBYurDC/zXDrPbDdVCYfeU0BsWO/8tqtlbgT2G9w84FoVxp7Z8VlI\n"
-"MCFlA2zs6SFz7JsDoeA3raAVGI/6ugLOpyypEBMs1OUIJqsil2D4kF501KKaU73yqWjgom7C\n"
-"12yxow+ev+to51byrvLjKzg6CYG1a4XXvi3tPxq3smPi9WIsgtRqAEFQ8TmDn5XpNpaYbg==\n"
-"-----END CERTIFICATE-----",
-
 /* QuoVadis Root CA 2 */
 "-----BEGIN CERTIFICATE-----\n"
 "MIIFtzCCA5+gAwIBAgICBQkwDQYJKoZIhvcNAQEFBQAwRTELMAkGA1UEBhMCQk0xGTAXBgNV\n"
@@ -184,78 +94,6 @@
 "zTSMmfXK4SVhM7JZG+Ju1zdXtg2pEto=\n"
 "-----END CERTIFICATE-----",
 
-/* XRamp Global CA Root */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIEMDCCAxigAwIBAgIQUJRs7Bjq1ZxN1ZfvdY+grTANBgkqhkiG9w0BAQUFADCBgjELMAkG\n"
-"A1UEBhMCVVMxHjAcBgNVBAsTFXd3dy54cmFtcHNlY3VyaXR5LmNvbTEkMCIGA1UEChMbWFJh\n"
-"bXAgU2VjdXJpdHkgU2VydmljZXMgSW5jMS0wKwYDVQQDEyRYUmFtcCBHbG9iYWwgQ2VydGlm\n"
-"aWNhdGlvbiBBdXRob3JpdHkwHhcNMDQxMTAxMTcxNDA0WhcNMzUwMTAxMDUzNzE5WjCBgjEL\n"
-"MAkGA1UEBhMCVVMxHjAcBgNVBAsTFXd3dy54cmFtcHNlY3VyaXR5LmNvbTEkMCIGA1UEChMb\n"
-"WFJhbXAgU2VjdXJpdHkgU2VydmljZXMgSW5jMS0wKwYDVQQDEyRYUmFtcCBHbG9iYWwgQ2Vy\n"
-"dGlmaWNhdGlvbiBBdXRob3JpdHkwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCY\n"
-"JB69FbS638eMpSe2OAtp87ZOqCwuIR1cRN8hXX4jdP5efrRKt6atH67gBhbim1vZZ3RrXYCP\n"
-"KZ2GG9mcDZhtdhAoWORlsH9KmHmf4MMxfoArtYzAQDsRhtDLooY2YKTVMIJt2W7QDxIEM5df\n"
-"T2Fa8OT5kavnHTu86M/0ay00fOJIYRyO82FEzG+gSqmUsE3a56k0enI4qEHMPJQRfevIpoy3\n"
-"hsvKMzvZPTeL+3o+hiznc9cKV6xkmxnr9A8ECIqsAxcZZPRaJSKNNCyy9mgdEm3Tih4U2sSP\n"
-"puIjhdV6Db1q4Ons7Be7QhtnqiXtRYMh/MHJfNViPvryxS3T/dRlAgMBAAGjgZ8wgZwwEwYJ\n"
-"KwYBBAGCNxQCBAYeBABDAEEwCwYDVR0PBAQDAgGGMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0O\n"
-"BBYEFMZPoj0GY4QJnM5i5ASsjVy16bYbMDYGA1UdHwQvMC0wK6ApoCeGJWh0dHA6Ly9jcmwu\n"
-"eHJhbXBzZWN1cml0eS5jb20vWEdDQS5jcmwwEAYJKwYBBAGCNxUBBAMCAQEwDQYJKoZIhvcN\n"
-"AQEFBQADggEBAJEVOQMBG2f7Shz5CmBbodpNl2L5JFMn14JkTpAuw0kbK5rc/Kh4ZzXxHfAR\n"
-"vbdI4xD2Dd8/0sm2qlWkSLoC295ZLhVbO50WfUfXN+pfTXYSNrsf16GBBEYgoyxtqZ4Bfj8p\n"
-"zgCT3/3JknOJiWSe5yvkHJEs0rnOfc5vMZnT5r7SHpDwCRR5XCOrTdLaIR9NmXmd4c8nnxCb\n"
-"HIgNsIpkQTG4DmyQJKSbXHGPurt+HBvbaoAPIbzp26a3QPSyi6mx5O+aGtA9aZnuqCij4Tyz\n"
-"8LIRnM98QObd50N9otg6tamN8jSZxNQQ4Qb9CYQQO+7ETPTsJ3xCwnR8gooJybQDJbw=\n"
-"-----END CERTIFICATE-----",
-
-/* Go Daddy Class 2 CA */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIEADCCAuigAwIBAgIBADANBgkqhkiG9w0BAQUFADBjMQswCQYDVQQGEwJVUzEhMB8GA1UE\n"
-"ChMYVGhlIEdvIERhZGR5IEdyb3VwLCBJbmMuMTEwLwYDVQQLEyhHbyBEYWRkeSBDbGFzcyAy\n"
-"IENlcnRpZmljYXRpb24gQXV0aG9yaXR5MB4XDTA0MDYyOTE3MDYyMFoXDTM0MDYyOTE3MDYy\n"
-"MFowYzELMAkGA1UEBhMCVVMxITAfBgNVBAoTGFRoZSBHbyBEYWRkeSBHcm91cCwgSW5jLjEx\n"
-"MC8GA1UECxMoR28gRGFkZHkgQ2xhc3MgMiBDZXJ0aWZpY2F0aW9uIEF1dGhvcml0eTCCASAw\n"
-"DQYJKoZIhvcNAQEBBQADggENADCCAQgCggEBAN6d1+pXGEmhW+vXX0iG6r7d/+TvZxz0ZWiz\n"
-"V3GgXne77ZtJ6XCAPVYYYwhv2vLM0D9/AlQiVBDYsoHUwHU9S3/Hd8M+eKsaA7Ugay9qK7HF\n"
-"iH7Eux6wwdhFJ2+qN1j3hybX2C32qRe3H3I2TqYXP2WYktsqbl2i/ojgC95/5Y0V4evLOtXi\n"
-"EqITLdiOr18SPaAIBQi2XKVlOARFmR6jYGB0xUGlcmIbYsUfb18aQr4CUWWoriMYavx4A6lN\n"
-"f4DD+qta/KFApMoZFv6yyO9ecw3ud72a9nmYvLEHZ6IVDd2gWMZEewo+YihfukEHU1jPEX44\n"
-"dMX4/7VpkI+EdOqXG68CAQOjgcAwgb0wHQYDVR0OBBYEFNLEsNKR1EwRcbNhyz2h/t2oatTj\n"
-"MIGNBgNVHSMEgYUwgYKAFNLEsNKR1EwRcbNhyz2h/t2oatTjoWekZTBjMQswCQYDVQQGEwJV\n"
-"UzEhMB8GA1UEChMYVGhlIEdvIERhZGR5IEdyb3VwLCBJbmMuMTEwLwYDVQQLEyhHbyBEYWRk\n"
-"eSBDbGFzcyAyIENlcnRpZmljYXRpb24gQXV0aG9yaXR5ggEAMAwGA1UdEwQFMAMBAf8wDQYJ\n"
-"KoZIhvcNAQEFBQADggEBADJL87LKPpH8EsahB4yOd6AzBhRckB4Y9wimPQoZ+YeAEW5p5JYX\n"
-"MP80kWNyOO7MHAGjHZQopDH2esRU1/blMVgDoszOYtuURXO1v0XJJLXVggKtI3lpjbi2Tc7P\n"
-"TMozI+gciKqdi0FuFskg5YmezTvacPd+mSYgFFQlq25zheabIZ0KbIIOqPjCDPoQHmyW74cN\n"
-"xA9hi63ugyuV+I6ShHI56yDqg+2DzZduCLzrTia2cyvk0/ZM/iZx4mERdEr/VxqHD3VILs9R\n"
-"aRegAhJhldXRQLIQTO7ErBBDpqWeCtWVYpoNz4iCxTIM5CufReYNnyicsbkqWletNw+vHX/b\n"
-"vZ8=\n"
-"-----END CERTIFICATE-----",
-
-/* Starfield Class 2 CA */
-"-----BEGIN CERTIFICATE-----\n"
-"MIIEDzCCAvegAwIBAgIBADANBgkqhkiG9w0BAQUFADBoMQswCQYDVQQGEwJVUzElMCMGA1UE\n"
-"ChMcU3RhcmZpZWxkIFRlY2hub2xvZ2llcywgSW5jLjEyMDAGA1UECxMpU3RhcmZpZWxkIENs\n"
-"YXNzIDIgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkwHhcNMDQwNjI5MTczOTE2WhcNMzQwNjI5\n"
-"MTczOTE2WjBoMQswCQYDVQQGEwJVUzElMCMGA1UEChMcU3RhcmZpZWxkIFRlY2hub2xvZ2ll\n"
-"cywgSW5jLjEyMDAGA1UECxMpU3RhcmZpZWxkIENsYXNzIDIgQ2VydGlmaWNhdGlvbiBBdXRo\n"
-"b3JpdHkwggEgMA0GCSqGSIb3DQEBAQUAA4IBDQAwggEIAoIBAQC3Msj+6XGmBIWtDBFk385N\n"
-"78gDGIc/oav7PKaf8MOh2tTYbitTkPskpD6E8J7oX+zlJ0T1KKY/e97gKvDIr1MvnsoFAZMe\n"
-"j2YcOadN+lq2cwQlZut3f+dZxkqZJRRU6ybH838Z1TBwj6+wRir/resp7defqgSHo9T5iaU0\n"
-"X9tDkYI22WY8sbi5gv2cOj4QyDvvBmVmepsZGD3/cVE8MC5fvj13c7JdBmzDI1aaK4Umkhyn\n"
-"ArPkPw2vCHmCuDY96pzTNbO8acr1zJ3o/WSNF4Azbl5KXZnJHoe0nRrA1W4TNSNe35tfPe/W\n"
-"93bC6j67eA0cQmdrBNj41tpvi/JEoAGrAgEDo4HFMIHCMB0GA1UdDgQWBBS/X7fRzt0fhvRb\n"
-"Vazc1xDCDqmI5zCBkgYDVR0jBIGKMIGHgBS/X7fRzt0fhvRbVazc1xDCDqmI56FspGowaDEL\n"
-"MAkGA1UEBhMCVVMxJTAjBgNVBAoTHFN0YXJmaWVsZCBUZWNobm9sb2dpZXMsIEluYy4xMjAw\n"
-"BgNVBAsTKVN0YXJmaWVsZCBDbGFzcyAyIENlcnRpZmljYXRpb24gQXV0aG9yaXR5ggEAMAwG\n"
-"A1UdEwQFMAMBAf8wDQYJKoZIhvcNAQEFBQADggEBAAWdP4id0ckaVaGsafPzWdqbAYcaT1ep\n"
-"oXkJKtv3L7IezMdeatiDh6GX70k1PncGQVhiv45YuApnP+yz3SFmH8lU+nLMPUxA2IGvd56D\n"
-"eruix/U0F47ZEUD0/CwqTRV/p2JdLiXTAAsgGh1o+Re49L2L7ShZ3U0WixeDyLJlxy16paq8\n"
-"U4Zt3VekyvggQQto8PT7dL5WXXp59fkdheMtlb71cZBDzI0fmgAKhynpVSJYACPq4xJDKVtH\n"
-"CN2MQWplBqjlIapBtJUhlbl90TSrE9atvNziPTnNvT51cKEYWQPJIrSPnNVeKtelttQKbfi3\n"
-"QBFGmh95DmK/D5fs4C8fF5Q=\n"
-"-----END CERTIFICATE-----",
-
 /* DigiCert Assured ID Root CA */
 "-----BEGIN CERTIFICATE-----\n"
 "MIIDtzCCAp+gAwIBAgIQDOfg5RfYRv6P5WD8G/AwOTANBgkqhkiG9w0BAQUFADBlMQswCQYD\n"
@@ -3598,6 +3436,52 @@
 "GJCy89UDyibK79XH4I9TjvAA46jtn/mtd+ArY0+ew+43u3gJhJ65bvspmZDogNOfJA==\n"
 "-----END CERTIFICATE-----",
 
+/* TrustAsia TLS ECC Root CA */
+"-----BEGIN CERTIFICATE-----\n"
+"MIICMTCCAbegAwIBAgIUNnThTXxlE8msg1UloD5Sfi9QaMcwCgYIKoZIzj0EAwMwWDELMAkG\n"
+"A1UEBhMCQ04xJTAjBgNVBAoTHFRydXN0QXNpYSBUZWNobm9sb2dpZXMsIEluYy4xIjAgBgNV\n"
+"BAMTGVRydXN0QXNpYSBUTFMgRUNDIFJvb3QgQ0EwHhcNMjQwNTE1MDU0MTU2WhcNNDQwNTE1\n"
+"MDU0MTU1WjBYMQswCQYDVQQGEwJDTjElMCMGA1UEChMcVHJ1c3RBc2lhIFRlY2hub2xvZ2ll\n"
+"cywgSW5jLjEiMCAGA1UEAxMZVHJ1c3RBc2lhIFRMUyBFQ0MgUm9vdCBDQTB2MBAGByqGSM49\n"
+"AgEGBSuBBAAiA2IABLh/pVs/AT598IhtrimY4ZtcU5nb9wj/1WrgjstEpvDBjL1P1M7UiFPo\n"
+"XlfXTr4sP/MSpwDpguMqWzJ8S5sUKZ74LYO1644xST0mYekdcouJtgq7nDM1D9rs3qlKH8kz\n"
+"saNCMEAwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQULIVTu7FDzTLqnqOH/qKYqKaT6RAw\n"
+"DgYDVR0PAQH/BAQDAgEGMAoGCCqGSM49BAMDA2gAMGUCMFRH18MtYYZI9HlaVQ01L18N9mds\n"
+"d0AaRuf4aFtOJx24mH1/k78ITcTaRTChD15KeAIxAKORh/IRM4PDwYqROkwrULG9IpRdNYlz\n"
+"g8WbGf60oenUoWa2AaU2+dhoYSi3dOGiMQ==\n"
+"-----END CERTIFICATE-----",
+
+/* TrustAsia TLS RSA Root CA */
+"-----BEGIN CERTIFICATE-----\n"
+"MIIFgDCCA2igAwIBAgIUHBjYz+VTPyI1RlNUJDxsR9FcSpwwDQYJKoZIhvcNAQEMBQAwWDEL\n"
+"MAkGA1UEBhMCQ04xJTAjBgNVBAoTHFRydXN0QXNpYSBUZWNobm9sb2dpZXMsIEluYy4xIjAg\n"
+"BgNVBAMTGVRydXN0QXNpYSBUTFMgUlNBIFJvb3QgQ0EwHhcNMjQwNTE1MDU0MTU3WhcNNDQw\n"
+"NTE1MDU0MTU2WjBYMQswCQYDVQQGEwJDTjElMCMGA1UEChMcVHJ1c3RBc2lhIFRlY2hub2xv\n"
+"Z2llcywgSW5jLjEiMCAGA1UEAxMZVHJ1c3RBc2lhIFRMUyBSU0EgUm9vdCBDQTCCAiIwDQYJ\n"
+"KoZIhvcNAQEBBQADggIPADCCAgoCggIBAMMWuBtqpERz5dZO9LnPWwvB0ZqB9WOwj0PBuwha\n"
+"GnrhB3YmH49pVr7+NmDQDIPNlOrnxS1cLwUWAp4KqC/lYCZUlviYQB2srp10Zy9U+5RjmOMm\n"
+"SoPGlbYJQ1DNDX3eRA5gEk9bNb2/mThtfWza4mhzH/kxpRkQcwUqwzIZheo0qt1CHjCNP561\n"
+"HmHVb70AcnKtEj+qpklz8oYVlQwQX1Fkzv93uMltrOXVmPGZLmzjyUT5tUMnCE32ft5Eebuy\n"
+"jBza00tsLtbDeLdM1aTk2tyKjg7/D8OmYCYozza/+lcK7Fs/6TAWe8TbxNRkoDD75f0dcZLd\n"
+"KY9BWN4ArTr9PXwaqLEX8E40eFgl1oUh63kd0Nyrz2I8sMeXi9bQn9P+PN7F4/w6g3CEIR0J\n"
+"wqH8uyghZVNgepBtljhb//HXeltt08lwSUq6HTrQUNoyIBnkiz/r1RYmNzz7dZ6wB3C4FGB3\n"
+"3PYPXFIKvF1tjVEK2sUYyJtt3LCDs3+jTnhMmCWr8n4uIF6CFabW2I+s5c0yhsj55NqJ4js+\n"
+"k8UTav/H9xj8Z7XvGCxUq0DTbE3txci3OE9kxJRMT6DNrqXGJyV1J23G2pyOsAWZ1SgRxSHU\n"
+"uPzHlqtKZFlhaxP8S8ySpg+kUb8OWJDZgoM5pl+z+m6Ss80zDoWo8SnTq1mt1tve1CuBAgMB\n"
+"AAGjQjBAMA8GA1UdEwEB/wQFMAMBAf8wHQYDVR0OBBYEFLgHkXlcBvRG/XtZylomkadFK/hT\n"
+"MA4GA1UdDwEB/wQEAwIBBjANBgkqhkiG9w0BAQwFAAOCAgEAIZtqBSBdGBanEqT3Rz/Nyjuu\n"
+"jsCCztxIJXgXbODgcMTWltnZ9r96nBO7U5WS/8+S4PPFJzVXqDuiGev4iqME3mmL5Dw8veWv\n"
+"0BIb5Ylrc5tvJQJLkIKvQMKtuppgJFqBTQUYo+IzeXoLH5Pt7DlK9RME7I10nYEKqG/odv6L\n"
+"TytpEoYKNDbdgptvT+Bz3Ul/KD7JO6NXBNiT2Twp2xIQaOHEibgGIOcberyxk2GaGUARtWqF\n"
+"VwHxtlotJnMnlvm5P1vQiJ3koP26TpUJg3933FEFlJ0gcXax7PqJtZwuhfG5WyRasQmr2soa\n"
+"B82G39tp27RIGAAtvKLEiUUjpQ7hRGU+isFqMB3iYPg6qocJQrmBktwliJiJ8Xw18WLK7nn4\n"
+"GS/+X/jbh87qqA8MpugLoDzga5SYnH+tBuYc6kIQX+ImFTw3OffXvO645e8D7r0i+yiGNFjE\n"
+"Wn9hongPXvPKnbwbPKfILfanIhHKA9jnZwqKDss1jjQ52MjqjZ9k4DewbNfFj8GQYSbbJIwe\n"
+"SsCI3zWQzj8C9GRh3sfIB5XeMhg6j6JCQCTl1jNdfK7vsU1P1FeQNWrcrgSXSYk0ly4wBOeY\n"
+"99sLAZDBHwo/+ML+TvrbmnNzFrwFuHnYWa8G5z9nODmxfKuU4CkUpijy323imttUQ/hHWKNd\n"
+"dBWcwauwxzQ=\n"
+"-----END CERTIFICATE-----",
+
 /* D-TRUST EV Root CA 2 2023 */
 "-----BEGIN CERTIFICATE-----\n"
 "MIIFqTCCA5GgAwIBAgIQaSYJfoBLTKCnjHhiU19abzANBgkqhkiG9w0BAQ0FADBIMQswCQYD\n"
@@ -3628,4 +3512,35 @@
 "S5N5YHVpD/Aa1VP6IQzCP+k/HxiMkl14p3ZnGbuy6n/pcAlWVqOwDAstNl7F6cTVg8uGF5cs\n"
 "bBNvh1qvSaYd2804BC5f4ko1Di1L+KIkBI3Y4WNeApI02phhXBxvWHZks/wCuPWdCg==\n"
 "-----END CERTIFICATE-----",
+
+/* SwissSign RSA TLS Root CA 2022 - 1 */
+"-----BEGIN CERTIFICATE-----\n"
+"MIIFkzCCA3ugAwIBAgIUQ/oMX04bgBhE79G0TzUfRPSA7cswDQYJKoZIhvcNAQELBQAwUTEL\n"
+"MAkGA1UEBhMCQ0gxFTATBgNVBAoTDFN3aXNzU2lnbiBBRzErMCkGA1UEAxMiU3dpc3NTaWdu\n"
+"IFJTQSBUTFMgUm9vdCBDQSAyMDIyIC0gMTAeFw0yMjA2MDgxMTA4MjJaFw00NzA2MDgxMTA4\n"
+"MjJaMFExCzAJBgNVBAYTAkNIMRUwEwYDVQQKEwxTd2lzc1NpZ24gQUcxKzApBgNVBAMTIlN3\n"
+"aXNzU2lnbiBSU0EgVExTIFJvb3QgQ0EgMjAyMiAtIDEwggIiMA0GCSqGSIb3DQEBAQUAA4IC\n"
+"DwAwggIKAoICAQDLKmjiC8NXvDVjvHClO/OMPE5Xlm7DTjak9gLKHqquuN6orx122ro10JFw\n"
+"B9+zBvKK8i5VUXu7LCTLf5ImgKO0lPaCoaTo+nUdWfMHamFk4saMla+ju45vVs9xzF6BYQ1t\n"
+"8qsCLqSX5XH8irCRIFucdFJtrhUnWXjyCcplDn/L9Ovn3KlMd/YrFgSVrpxxpT8q2kFC5zyE\n"
+"EPThPYxr4iuRR1VPuFa+Rd4iUU1OKNlfGUEGjw5NBuBwQCMBauTLE5tzrE0USJIt/m2n+Idr\n"
+"eXXhvhCxqohAWVTXz8TQm0SzOGlkjIHRI36qOTw7D59Ke4LKa2/KIj4x0LDQKhySio/YGZxH\n"
+"5D4MucLNvkEM+KRHBdvBFzA4OmnczcNpI/2aDwLOEGrOyvi5KaM2iYauC8BPY7kGWUleDsFp\n"
+"swrzd34unYyzJ5jSmY0lpx+Gs6ZUcDj8fV3oT4MM0ZPlEuRU2j7yrTrePjxF8CgPBrnh25d7\n"
+"mUWe3f6VWQQvdT/TromZhqwUtKiE+shdOxtYk8EXlFXIC+OCeYSf8wCENO7cMdWP8vpPlkwG\n"
+"qnj73mSiI80fPsWMvDdUDrtaclXvyFu1cvh43zcgTFeRc5JzrBh3Q4IgaezprClG5QtO+Ddz\n"
+"iZaKHG29777YtvTKwP1H8K4LWCDFyB02rpeNUIMmJCn3nTsPBQIDAQABo2MwYTAPBgNVHRMB\n"
+"Af8EBTADAQH/MA4GA1UdDwEB/wQEAwIBBjAfBgNVHSMEGDAWgBRvjmKLk0Ow4UD2p8P98Q+4\n"
+"DxU4pTAdBgNVHQ4EFgQUb45ii5NDsOFA9qfD/fEPuA8VOKUwDQYJKoZIhvcNAQELBQADggIB\n"
+"AKwsKUF9+lz1GpUYvyypiqkkVHX1uECry6gkUSsYP2OprphWKwVDIqO310aewCoSPY6WlkDf\n"
+"DDOLazeROpW7OSltwAJsipQLBwJNGD77+3v1dj2b9l4wBlgzHqp41eZUBDqyggmNzhYzWUUo\n"
+"8aWjlw5DI/0LIICQ/+Mmz7hkkeUFjxOgdg3XNwwQiJb0Pr6VvfHDffCjw3lHC1ySFWPtUnWK\n"
+"50Zpy1FVCypM9fJkT6lc/2cyjlUtMoIcgC9qkfjLvH4YoiaoLqNTKIftV+Vlek4ASltOU8li\n"
+"Nr3CjlvrzG4ngRhZi0Rjn9UMZfQpZX+RLOV/fuiJz48gy20HQhFRJjKKLjpHE7iNvUcNCfAW\n"
+"pO2Whi4Z2L6MOuhFLhG6rlrnub+xzI/goP+4s9GFe3lmozm1O2bYQL7Pt2eLSMkZJVX8vY3P\n"
+"XtpOpvJpzv1/THfQwUY1mFwjmwJFQ5Ra3bxHrSL+ul4vkSkphnsh3m5kt8sNjzdbowhq6/Td\n"
+"Ao9QAwKxuDdollDruF/UKIqlIgyKhPBZLtU30WHlQnNYKoH3dtvi4k0NX/a3vgW0rk4N3hY9\n"
+"A4GzJl5LuEsAz/+MF7psYC0nhzck5npgL7XTgwSqT0N1osGDsieYK7EOgLrAhV5Cud+xYJHT\n"
+"6xh+cHiudoO+cVrQkOPKwRYlZ0rwtnu64ZzZ\n"
+"-----END CERTIFICATE-----",
 #endif  // defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
diff --git a/tools/certdata.txt b/tools/certdata.txt
index 1ed5a248b5..aeedd56c53 100644
--- a/tools/certdata.txt
+++ b/tools/certdata.txt
@@ -200,7 +200,7 @@ END
 CKA_SERIAL_NUMBER MULTILINE_OCTAL
 \002\013\004\000\000\000\000\001\025\113\132\303\224
 END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
@@ -366,141 +366,7 @@ END
 CKA_SERIAL_NUMBER MULTILINE_OCTAL
 \002\004\070\143\336\370
 END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
-#
-# Certificate "Baltimore CyberTrust Root"
-#
-# Issuer: CN=Baltimore CyberTrust Root,OU=CyberTrust,O=Baltimore,C=IE
-# Serial Number: 33554617 (0x20000b9)
-# Subject: CN=Baltimore CyberTrust Root,OU=CyberTrust,O=Baltimore,C=IE
-# Not Valid Before: Fri May 12 18:46:00 2000
-# Not Valid After : Mon May 12 23:59:00 2025
-# Fingerprint (SHA-256): 16:AF:57:A9:F6:76:B0:AB:12:60:95:AA:5E:BA:DE:F2:2A:B3:11:19:D6:44:AC:95:CD:4B:93:DB:F3:F2:6A:EB
-# Fingerprint (SHA1): D4:DE:20:D0:5E:66:FC:53:FE:1A:50:88:2C:78:DB:28:52:CA:E4:74
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "Baltimore CyberTrust Root"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\111\105\061
-\022\060\020\006\003\125\004\012\023\011\102\141\154\164\151\155
-\157\162\145\061\023\060\021\006\003\125\004\013\023\012\103\171
-\142\145\162\124\162\165\163\164\061\042\060\040\006\003\125\004
-\003\023\031\102\141\154\164\151\155\157\162\145\040\103\171\142
-\145\162\124\162\165\163\164\040\122\157\157\164
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\111\105\061
-\022\060\020\006\003\125\004\012\023\011\102\141\154\164\151\155
-\157\162\145\061\023\060\021\006\003\125\004\013\023\012\103\171
-\142\145\162\124\162\165\163\164\061\042\060\040\006\003\125\004
-\003\023\031\102\141\154\164\151\155\157\162\145\040\103\171\142
-\145\162\124\162\165\163\164\040\122\157\157\164
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\004\002\000\000\271
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\003\167\060\202\002\137\240\003\002\001\002\002\004\002
-\000\000\271\060\015\006\011\052\206\110\206\367\015\001\001\005
-\005\000\060\132\061\013\060\011\006\003\125\004\006\023\002\111
-\105\061\022\060\020\006\003\125\004\012\023\011\102\141\154\164
-\151\155\157\162\145\061\023\060\021\006\003\125\004\013\023\012
-\103\171\142\145\162\124\162\165\163\164\061\042\060\040\006\003
-\125\004\003\023\031\102\141\154\164\151\155\157\162\145\040\103
-\171\142\145\162\124\162\165\163\164\040\122\157\157\164\060\036
-\027\015\060\060\060\065\061\062\061\070\064\066\060\060\132\027
-\015\062\065\060\065\061\062\062\063\065\071\060\060\132\060\132
-\061\013\060\011\006\003\125\004\006\023\002\111\105\061\022\060
-\020\006\003\125\004\012\023\011\102\141\154\164\151\155\157\162
-\145\061\023\060\021\006\003\125\004\013\023\012\103\171\142\145
-\162\124\162\165\163\164\061\042\060\040\006\003\125\004\003\023
-\031\102\141\154\164\151\155\157\162\145\040\103\171\142\145\162
-\124\162\165\163\164\040\122\157\157\164\060\202\001\042\060\015
-\006\011\052\206\110\206\367\015\001\001\001\005\000\003\202\001
-\017\000\060\202\001\012\002\202\001\001\000\243\004\273\042\253
-\230\075\127\350\046\162\232\265\171\324\051\342\341\350\225\200
-\261\260\343\133\216\053\051\232\144\337\241\135\355\260\011\005
-\155\333\050\056\316\142\242\142\376\264\210\332\022\353\070\353
-\041\235\300\101\053\001\122\173\210\167\323\034\217\307\272\271
-\210\265\152\011\347\163\350\021\100\247\321\314\312\142\215\055
-\345\217\013\246\120\322\250\120\303\050\352\365\253\045\207\212
-\232\226\034\251\147\270\077\014\325\367\371\122\023\057\302\033
-\325\160\160\360\217\300\022\312\006\313\232\341\331\312\063\172
-\167\326\370\354\271\361\150\104\102\110\023\322\300\302\244\256
-\136\140\376\266\246\005\374\264\335\007\131\002\324\131\030\230
-\143\365\245\143\340\220\014\175\135\262\006\172\363\205\352\353
-\324\003\256\136\204\076\137\377\025\355\151\274\371\071\066\162
-\165\317\167\122\115\363\311\220\054\271\075\345\311\043\123\077
-\037\044\230\041\134\007\231\051\275\306\072\354\347\156\206\072
-\153\227\164\143\063\275\150\030\061\360\170\215\166\277\374\236
-\216\135\052\206\247\115\220\334\047\032\071\002\003\001\000\001
-\243\105\060\103\060\035\006\003\125\035\016\004\026\004\024\345
-\235\131\060\202\107\130\314\254\372\010\124\066\206\173\072\265
-\004\115\360\060\022\006\003\125\035\023\001\001\377\004\010\060
-\006\001\001\377\002\001\003\060\016\006\003\125\035\017\001\001
-\377\004\004\003\002\001\006\060\015\006\011\052\206\110\206\367
-\015\001\001\005\005\000\003\202\001\001\000\205\014\135\216\344
-\157\121\150\102\005\240\335\273\117\047\045\204\003\275\367\144
-\375\055\327\060\343\244\020\027\353\332\051\051\266\171\077\166
-\366\031\023\043\270\020\012\371\130\244\324\141\160\275\004\141
-\152\022\212\027\325\012\275\305\274\060\174\326\351\014\045\215
-\206\100\117\354\314\243\176\070\306\067\021\117\355\335\150\061
-\216\114\322\263\001\164\356\276\165\136\007\110\032\177\160\377
-\026\134\204\300\171\205\270\005\375\177\276\145\021\243\017\300
-\002\264\370\122\067\071\004\325\251\061\172\030\277\240\052\364
-\022\231\367\243\105\202\343\074\136\365\235\236\265\310\236\174
-\056\310\244\236\116\010\024\113\155\375\160\155\153\032\143\275
-\144\346\037\267\316\360\362\237\056\273\033\267\362\120\210\163
-\222\302\342\343\026\215\232\062\002\253\216\030\335\351\020\021
-\356\176\065\253\220\257\076\060\224\172\320\063\075\247\145\017
-\365\374\216\236\142\317\107\104\054\001\135\273\035\265\062\322
-\107\322\070\056\320\376\201\334\062\152\036\265\356\074\325\374
-\347\201\035\031\303\044\102\352\143\071\251
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "Baltimore CyberTrust Root"
-# Issuer: CN=Baltimore CyberTrust Root,OU=CyberTrust,O=Baltimore,C=IE
-# Serial Number: 33554617 (0x20000b9)
-# Subject: CN=Baltimore CyberTrust Root,OU=CyberTrust,O=Baltimore,C=IE
-# Not Valid Before: Fri May 12 18:46:00 2000
-# Not Valid After : Mon May 12 23:59:00 2025
-# Fingerprint (SHA-256): 16:AF:57:A9:F6:76:B0:AB:12:60:95:AA:5E:BA:DE:F2:2A:B3:11:19:D6:44:AC:95:CD:4B:93:DB:F3:F2:6A:EB
-# Fingerprint (SHA1): D4:DE:20:D0:5E:66:FC:53:FE:1A:50:88:2C:78:DB:28:52:CA:E4:74
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "Baltimore CyberTrust Root"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\324\336\040\320\136\146\374\123\376\032\120\210\054\170\333\050
-\122\312\344\164
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\254\266\224\245\234\027\340\327\221\122\233\261\227\006\246\344
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\111\105\061
-\022\060\020\006\003\125\004\012\023\011\102\141\154\164\151\155
-\157\162\145\061\023\060\021\006\003\125\004\013\023\012\103\171
-\142\145\162\124\162\165\163\164\061\042\060\040\006\003\125\004
-\003\023\031\102\141\154\164\151\155\157\162\145\040\103\171\142
-\145\162\124\162\165\163\164\040\122\157\157\164
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\004\002\000\000\271
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
@@ -946,7 +812,7 @@ END
 CKA_SERIAL_NUMBER MULTILINE_OCTAL
 \002\001\001
 END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
@@ -1452,7 +1318,7 @@ CKA_SERIAL_NUMBER MULTILINE_OCTAL
 \002\020\120\224\154\354\030\352\325\234\115\325\227\357\165\217
 \240\255
 END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
@@ -1598,8 +1464,8 @@ END
 CKA_SERIAL_NUMBER MULTILINE_OCTAL
 \002\001\000
 END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
@@ -1745,8 +1611,8 @@ END
 CKA_SERIAL_NUMBER MULTILINE_OCTAL
 \002\001\000
 END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
@@ -3323,9 +3189,13 @@ CKA_VALUE MULTILINE_OCTAL
 \201\370\021\234
 END
 CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+# For Server Distrust After: Tue Apr 15 23:59:59 2025
+CKA_NSS_SERVER_DISTRUST_AFTER MULTILINE_OCTAL
+\062\065\060\064\061\065\062\063\065\071\065\071\132
+END
 CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
 
+
 # Trust for "ePKI Root Certification Authority"
 # Issuer: OU=ePKI Root Certification Authority,O="Chunghwa Telecom Co., Ltd.",C=TW
 # Serial Number:15:c8:bd:65:47:5c:af:b8:97:00:5e:e4:06:d2:bc:9d
@@ -25971,6 +25841,576 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
+#
+# Certificate "TrustAsia SMIME ECC Root CA"
+#
+# Issuer: CN=TrustAsia SMIME ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:5a:c2:f8:29:4f:e3:7d:c5:5e:1d:18:6f:3b:93:20:1f:ff:7b:ba:2d
+# Subject: CN=TrustAsia SMIME ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:41:59 2024
+# Not Valid After : Sun May 15 05:41:58 2044
+# Fingerprint (SHA-256): 43:64:72:C1:00:9A:32:5C:54:F1:A5:BB:B5:46:8A:7B:AE:EC:CB:E0:5D:E5:F0:99:CB:70:D3:FE:41:E1:3C:16
+# Fingerprint (SHA1): 8C:0D:AA:FE:13:FA:59:F2:DB:9D:0C:97:DA:12:A2:45:1A:02:13:54
+CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia SMIME ECC Root CA"
+CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+CKA_SUBJECT MULTILINE_OCTAL
+\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\023
+\033\124\162\165\163\164\101\163\151\141\040\123\115\111\115\105
+\040\105\103\103\040\122\157\157\164\040\103\101
+END
+CKA_ID UTF8 "0"
+CKA_ISSUER MULTILINE_OCTAL
+\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\023
+\033\124\162\165\163\164\101\163\151\141\040\123\115\111\115\105
+\040\105\103\103\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\132\302\370\051\117\343\175\305\136\035\030\157\073\223
+\040\037\377\173\272\055
+END
+CKA_VALUE MULTILINE_OCTAL
+\060\202\002\066\060\202\001\273\240\003\002\001\002\002\024\132
+\302\370\051\117\343\175\305\136\035\030\157\073\223\040\037\377
+\173\272\055\060\012\006\010\052\206\110\316\075\004\003\003\060
+\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061\045
+\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101\163
+\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163\054
+\040\111\156\143\056\061\044\060\042\006\003\125\004\003\023\033
+\124\162\165\163\164\101\163\151\141\040\123\115\111\115\105\040
+\105\103\103\040\122\157\157\164\040\103\101\060\036\027\015\062
+\064\060\065\061\065\060\065\064\061\065\071\132\027\015\064\064
+\060\065\061\065\060\065\064\061\065\070\132\060\132\061\013\060
+\011\006\003\125\004\006\023\002\103\116\061\045\060\043\006\003
+\125\004\012\023\034\124\162\165\163\164\101\163\151\141\040\124
+\145\143\150\156\157\154\157\147\151\145\163\054\040\111\156\143
+\056\061\044\060\042\006\003\125\004\003\023\033\124\162\165\163
+\164\101\163\151\141\040\123\115\111\115\105\040\105\103\103\040
+\122\157\157\164\040\103\101\060\166\060\020\006\007\052\206\110
+\316\075\002\001\006\005\053\201\004\000\042\003\142\000\004\315
+\331\373\047\275\151\354\206\311\220\103\261\160\027\224\247\311
+\167\043\072\077\043\145\323\034\243\035\036\102\124\040\353\320
+\102\273\131\271\213\254\362\206\215\113\216\035\252\226\042\043
+\071\166\155\222\233\353\042\313\172\103\242\163\375\163\113\302
+\045\017\052\261\151\306\377\105\337\251\074\225\200\273\251\036
+\112\223\057\354\034\035\210\364\021\222\067\237\067\230\162\243
+\102\060\100\060\017\006\003\125\035\023\001\001\377\004\005\060
+\003\001\001\377\060\035\006\003\125\035\016\004\026\004\024\061
+\147\346\162\262\015\346\042\240\254\317\176\042\247\131\062\343
+\146\043\245\060\016\006\003\125\035\017\001\001\377\004\004\003
+\002\001\006\060\012\006\010\052\206\110\316\075\004\003\003\003
+\151\000\060\146\002\061\000\335\072\114\215\244\306\177\355\275
+\245\306\117\076\375\061\113\050\326\212\126\337\145\026\167\207
+\115\373\272\062\010\201\340\236\063\110\213\237\224\206\357\001
+\053\224\346\214\326\324\216\002\061\000\237\201\234\260\046\375
+\053\326\362\365\161\204\236\250\307\212\214\326\130\021\122\265
+\270\004\313\314\161\165\143\342\306\031\070\331\155\154\031\161
+\244\026\031\041\223\272\006\104\050\154
+END
+CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
+CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+
+# Trust for "TrustAsia SMIME ECC Root CA"
+# Issuer: CN=TrustAsia SMIME ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:5a:c2:f8:29:4f:e3:7d:c5:5e:1d:18:6f:3b:93:20:1f:ff:7b:ba:2d
+# Subject: CN=TrustAsia SMIME ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:41:59 2024
+# Not Valid After : Sun May 15 05:41:58 2044
+# Fingerprint (SHA-256): 43:64:72:C1:00:9A:32:5C:54:F1:A5:BB:B5:46:8A:7B:AE:EC:CB:E0:5D:E5:F0:99:CB:70:D3:FE:41:E1:3C:16
+# Fingerprint (SHA1): 8C:0D:AA:FE:13:FA:59:F2:DB:9D:0C:97:DA:12:A2:45:1A:02:13:54
+CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia SMIME ECC Root CA"
+CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+\214\015\252\376\023\372\131\362\333\235\014\227\332\022\242\105
+\032\002\023\124
+END
+CKA_CERT_MD5_HASH MULTILINE_OCTAL
+\225\175\377\225\115\114\152\373\215\014\017\317\102\333\357\040
+END
+CKA_ISSUER MULTILINE_OCTAL
+\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\023
+\033\124\162\165\163\164\101\163\151\141\040\123\115\111\115\105
+\040\105\103\103\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\132\302\370\051\117\343\175\305\136\035\030\157\073\223
+\040\037\377\173\272\055
+END
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
+#
+# Certificate "TrustAsia SMIME RSA Root CA"
+#
+# Issuer: CN=TrustAsia SMIME RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:5a:ee:71:df:de:0c:57:85:b5:bb:36:22:d7:b8:76:46:02:73:ca:ff
+# Subject: CN=TrustAsia SMIME RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:42:01 2024
+# Not Valid After : Sun May 15 05:42:00 2044
+# Fingerprint (SHA-256): C7:79:6B:EB:62:C1:01:BB:14:3D:26:2A:7C:96:A0:C6:16:81:83:22:3E:F5:0D:69:96:32:D8:6E:03:B8:CC:9B
+# Fingerprint (SHA1): 8C:69:21:7C:CE:49:73:36:5A:5C:EF:2B:B0:86:97:50:E3:E3:33:65
+CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia SMIME RSA Root CA"
+CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+CKA_SUBJECT MULTILINE_OCTAL
+\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\023
+\033\124\162\165\163\164\101\163\151\141\040\123\115\111\115\105
+\040\122\123\101\040\122\157\157\164\040\103\101
+END
+CKA_ID UTF8 "0"
+CKA_ISSUER MULTILINE_OCTAL
+\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\023
+\033\124\162\165\163\164\101\163\151\141\040\123\115\111\115\105
+\040\122\123\101\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\132\356\161\337\336\014\127\205\265\273\066\042\327\270
+\166\106\002\163\312\377
+END
+CKA_VALUE MULTILINE_OCTAL
+\060\202\005\204\060\202\003\154\240\003\002\001\002\002\024\132
+\356\161\337\336\014\127\205\265\273\066\042\327\270\166\106\002
+\163\312\377\060\015\006\011\052\206\110\206\367\015\001\001\014
+\005\000\060\132\061\013\060\011\006\003\125\004\006\023\002\103
+\116\061\045\060\043\006\003\125\004\012\023\034\124\162\165\163
+\164\101\163\151\141\040\124\145\143\150\156\157\154\157\147\151
+\145\163\054\040\111\156\143\056\061\044\060\042\006\003\125\004
+\003\023\033\124\162\165\163\164\101\163\151\141\040\123\115\111
+\115\105\040\122\123\101\040\122\157\157\164\040\103\101\060\036
+\027\015\062\064\060\065\061\065\060\065\064\062\060\061\132\027
+\015\064\064\060\065\061\065\060\065\064\062\060\060\132\060\132
+\061\013\060\011\006\003\125\004\006\023\002\103\116\061\045\060
+\043\006\003\125\004\012\023\034\124\162\165\163\164\101\163\151
+\141\040\124\145\143\150\156\157\154\157\147\151\145\163\054\040
+\111\156\143\056\061\044\060\042\006\003\125\004\003\023\033\124
+\162\165\163\164\101\163\151\141\040\123\115\111\115\105\040\122
+\123\101\040\122\157\157\164\040\103\101\060\202\002\042\060\015
+\006\011\052\206\110\206\367\015\001\001\001\005\000\003\202\002
+\017\000\060\202\002\012\002\202\002\001\000\230\225\234\255\074
+\131\163\323\223\166\246\110\124\246\034\210\162\114\115\341\202
+\377\032\023\037\120\336\214\331\220\102\321\274\231\323\067\243
+\327\161\072\312\335\136\033\220\141\102\156\217\100\001\163\175
+\037\061\272\324\035\156\004\322\252\220\204\112\063\012\104\237
+\132\031\037\264\156\055\150\374\257\300\237\112\020\022\277\076
+\202\302\202\260\057\347\220\157\220\144\020\203\354\354\066\224
+\014\174\117\340\002\015\075\207\171\102\115\000\345\060\256\160
+\037\025\375\372\317\122\045\066\115\171\040\273\361\020\230\340
+\140\207\311\365\170\014\042\355\002\346\021\154\361\233\132\221
+\021\164\206\320\376\170\222\226\004\303\230\357\306\160\257\251
+\220\252\026\366\042\355\330\372\106\050\161\342\311\010\300\120
+\274\166\243\272\055\226\362\021\371\244\275\020\126\370\047\155
+\053\113\144\210\334\027\135\317\000\010\212\222\256\032\314\022
+\327\337\360\020\050\363\315\072\352\072\022\214\033\347\141\011
+\103\152\344\070\351\376\164\071\333\315\012\073\303\117\123\326
+\256\161\325\070\203\207\037\316\347\114\363\021\125\207\337\267
+\257\367\353\032\206\020\155\250\355\110\335\062\320\333\252\052
+\013\162\167\051\167\375\172\350\077\170\040\052\150\276\336\157
+\010\346\265\205\107\200\067\371\161\226\213\245\230\126\227\174
+\021\145\026\216\324\316\330\047\324\371\132\305\247\215\273\321
+\067\015\245\253\145\135\332\164\223\210\136\333\165\366\177\233
+\302\232\261\155\010\002\014\230\112\273\315\060\257\166\065\002
+\205\126\043\121\011\100\011\005\367\360\075\036\302\310\172\032
+\272\305\101\025\271\015\166\264\264\244\270\105\375\215\243\073
+\075\003\052\315\263\225\177\064\173\335\241\117\331\072\377\047
+\365\015\302\001\234\377\131\165\132\032\311\347\322\375\177\117
+\017\265\376\015\074\346\240\326\373\265\166\134\264\060\115\020
+\077\376\342\353\132\016\261\117\117\120\176\010\063\002\221\127
+\062\111\211\247\100\020\120\027\153\237\143\272\262\262\126\230
+\166\364\377\162\351\004\234\324\247\114\117\366\072\037\132\124
+\061\157\313\272\062\255\370\155\257\105\307\173\143\104\337\302
+\375\170\350\112\212\260\164\165\026\336\217\002\003\001\000\001
+\243\102\060\100\060\017\006\003\125\035\023\001\001\377\004\005
+\060\003\001\001\377\060\035\006\003\125\035\016\004\026\004\024
+\200\032\252\103\301\311\177\213\032\226\105\274\075\273\153\110
+\122\154\133\253\060\016\006\003\125\035\017\001\001\377\004\004
+\003\002\001\006\060\015\006\011\052\206\110\206\367\015\001\001
+\014\005\000\003\202\002\001\000\052\165\201\241\202\040\352\177
+\126\256\011\060\227\020\170\364\331\100\024\122\244\356\152\177
+\360\011\050\326\020\143\340\116\173\311\061\307\010\245\027\353
+\257\206\342\227\007\117\251\016\242\205\314\355\244\200\201\035
+\132\123\006\074\114\032\251\267\204\206\317\174\365\205\012\160
+\231\320\355\066\221\215\347\167\056\316\204\234\050\206\055\045
+\065\214\114\001\346\175\333\141\135\342\167\204\323\001\321\142
+\335\014\242\012\144\244\374\360\355\072\110\362\071\354\065\326
+\241\172\027\161\354\323\322\354\270\046\373\255\025\160\126\112
+\317\144\300\271\140\133\367\346\315\240\302\003\053\371\262\345
+\332\272\245\067\034\330\215\332\171\164\052\147\344\242\172\307
+\054\215\245\301\152\201\154\314\365\251\071\152\042\175\242\342
+\010\315\351\223\333\272\313\037\165\167\236\367\230\257\024\344
+\073\240\114\247\162\347\265\136\123\111\314\377\222\127\361\131
+\053\011\272\351\122\223\300\017\105\013\141\326\154\131\163\013
+\330\371\001\057\332\003\117\067\034\364\337\203\136\013\101\240
+\261\174\071\000\241\154\320\330\264\221\162\053\033\077\242\232
+\321\304\337\053\325\310\225\175\204\315\166\033\334\344\044\306
+\077\025\042\075\205\247\102\053\355\176\161\131\141\110\302\075
+\221\143\100\174\051\263\200\026\012\045\374\030\140\016\123\255
+\016\052\316\374\101\072\315\261\344\152\153\314\071\155\310\115
+\145\270\252\223\210\170\156\017\002\213\340\226\216\164\037\320
+\176\341\164\267\366\201\152\344\043\254\216\375\217\065\035\232
+\161\012\117\052\125\057\376\030\301\000\367\173\110\212\145\257
+\157\010\035\103\347\132\270\037\244\350\003\353\375\114\265\264
+\223\250\061\336\103\235\022\275\347\351\244\322\337\157\374\157
+\106\243\357\061\116\215\027\253\033\251\340\370\251\064\126\241
+\016\347\164\354\250\207\205\330\167\065\113\215\076\156\162\174
+\110\314\037\162\263\161\164\324\044\206\274\310\300\343\171\024
+\001\316\331\372\201\306\134\263\371\135\014\070\276\316\362\216
+\215\137\075\045\341\121\071\043\250\352\000\216\004\263\277\162
+\240\302\272\203\373\315\244\227\165\040\232\277\247\120\342\335
+\255\054\037\364\077\243\310\061
+END
+CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
+CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+
+# Trust for "TrustAsia SMIME RSA Root CA"
+# Issuer: CN=TrustAsia SMIME RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:5a:ee:71:df:de:0c:57:85:b5:bb:36:22:d7:b8:76:46:02:73:ca:ff
+# Subject: CN=TrustAsia SMIME RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:42:01 2024
+# Not Valid After : Sun May 15 05:42:00 2044
+# Fingerprint (SHA-256): C7:79:6B:EB:62:C1:01:BB:14:3D:26:2A:7C:96:A0:C6:16:81:83:22:3E:F5:0D:69:96:32:D8:6E:03:B8:CC:9B
+# Fingerprint (SHA1): 8C:69:21:7C:CE:49:73:36:5A:5C:EF:2B:B0:86:97:50:E3:E3:33:65
+CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia SMIME RSA Root CA"
+CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+\214\151\041\174\316\111\163\066\132\134\357\053\260\206\227\120
+\343\343\063\145
+END
+CKA_CERT_MD5_HASH MULTILINE_OCTAL
+\151\206\230\353\337\305\241\132\301\244\153\324\074\243\373\266
+END
+CKA_ISSUER MULTILINE_OCTAL
+\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\023
+\033\124\162\165\163\164\101\163\151\141\040\123\115\111\115\105
+\040\122\123\101\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\132\356\161\337\336\014\127\205\265\273\066\042\327\270
+\166\106\002\163\312\377
+END
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
+#
+# Certificate "TrustAsia TLS ECC Root CA"
+#
+# Issuer: CN=TrustAsia TLS ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:36:74:e1:4d:7c:65:13:c9:ac:83:55:25:a0:3e:52:7e:2f:50:68:c7
+# Subject: CN=TrustAsia TLS ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:41:56 2024
+# Not Valid After : Sun May 15 05:41:55 2044
+# Fingerprint (SHA-256): C0:07:6B:9E:F0:53:1F:B1:A6:56:D6:7C:4E:BE:97:CD:5D:BA:A4:1E:F4:45:98:AC:C2:48:98:78:C9:2D:87:11
+# Fingerprint (SHA1): B5:EC:39:F3:A1:66:37:AE:C3:05:94:57:E2:BE:11:BE:B7:A1:7F:36
+CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia TLS ECC Root CA"
+CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+CKA_SUBJECT MULTILINE_OCTAL
+\060\130\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\042\060\040\006\003\125\004\003\023
+\031\124\162\165\163\164\101\163\151\141\040\124\114\123\040\105
+\103\103\040\122\157\157\164\040\103\101
+END
+CKA_ID UTF8 "0"
+CKA_ISSUER MULTILINE_OCTAL
+\060\130\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\042\060\040\006\003\125\004\003\023
+\031\124\162\165\163\164\101\163\151\141\040\124\114\123\040\105
+\103\103\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\066\164\341\115\174\145\023\311\254\203\125\045\240\076
+\122\176\057\120\150\307
+END
+CKA_VALUE MULTILINE_OCTAL
+\060\202\002\061\060\202\001\267\240\003\002\001\002\002\024\066
+\164\341\115\174\145\023\311\254\203\125\045\240\076\122\176\057
+\120\150\307\060\012\006\010\052\206\110\316\075\004\003\003\060
+\130\061\013\060\011\006\003\125\004\006\023\002\103\116\061\045
+\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101\163
+\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163\054
+\040\111\156\143\056\061\042\060\040\006\003\125\004\003\023\031
+\124\162\165\163\164\101\163\151\141\040\124\114\123\040\105\103
+\103\040\122\157\157\164\040\103\101\060\036\027\015\062\064\060
+\065\061\065\060\065\064\061\065\066\132\027\015\064\064\060\065
+\061\065\060\065\064\061\065\065\132\060\130\061\013\060\011\006
+\003\125\004\006\023\002\103\116\061\045\060\043\006\003\125\004
+\012\023\034\124\162\165\163\164\101\163\151\141\040\124\145\143
+\150\156\157\154\157\147\151\145\163\054\040\111\156\143\056\061
+\042\060\040\006\003\125\004\003\023\031\124\162\165\163\164\101
+\163\151\141\040\124\114\123\040\105\103\103\040\122\157\157\164
+\040\103\101\060\166\060\020\006\007\052\206\110\316\075\002\001
+\006\005\053\201\004\000\042\003\142\000\004\270\177\245\133\077
+\001\076\175\360\210\155\256\051\230\341\233\134\123\231\333\367
+\010\377\325\152\340\216\313\104\246\360\301\214\275\117\324\316
+\324\210\123\350\136\127\327\116\276\054\077\363\022\247\000\351
+\202\343\052\133\062\174\113\233\024\051\236\370\055\203\265\353
+\216\061\111\075\046\141\351\035\162\213\211\266\012\273\234\063
+\065\017\332\354\336\251\112\037\311\063\261\243\102\060\100\060
+\017\006\003\125\035\023\001\001\377\004\005\060\003\001\001\377
+\060\035\006\003\125\035\016\004\026\004\024\054\205\123\273\261
+\103\315\062\352\236\243\207\376\242\230\250\246\223\351\020\060
+\016\006\003\125\035\017\001\001\377\004\004\003\002\001\006\060
+\012\006\010\052\206\110\316\075\004\003\003\003\150\000\060\145
+\002\060\124\107\327\303\055\141\206\110\364\171\132\125\015\065
+\057\137\015\366\147\154\167\100\032\106\347\370\150\133\116\047
+\035\270\230\175\177\223\277\010\115\304\332\105\060\241\017\136
+\112\170\002\061\000\243\221\207\362\021\063\203\303\301\212\221
+\072\114\053\120\261\275\042\224\135\065\211\163\203\305\233\031
+\376\264\241\351\324\241\146\266\001\245\066\371\330\150\141\050
+\267\164\341\242\061
+END
+CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
+CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+
+# Trust for "TrustAsia TLS ECC Root CA"
+# Issuer: CN=TrustAsia TLS ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:36:74:e1:4d:7c:65:13:c9:ac:83:55:25:a0:3e:52:7e:2f:50:68:c7
+# Subject: CN=TrustAsia TLS ECC Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:41:56 2024
+# Not Valid After : Sun May 15 05:41:55 2044
+# Fingerprint (SHA-256): C0:07:6B:9E:F0:53:1F:B1:A6:56:D6:7C:4E:BE:97:CD:5D:BA:A4:1E:F4:45:98:AC:C2:48:98:78:C9:2D:87:11
+# Fingerprint (SHA1): B5:EC:39:F3:A1:66:37:AE:C3:05:94:57:E2:BE:11:BE:B7:A1:7F:36
+CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia TLS ECC Root CA"
+CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+\265\354\071\363\241\146\067\256\303\005\224\127\342\276\021\276
+\267\241\177\066
+END
+CKA_CERT_MD5_HASH MULTILINE_OCTAL
+\011\110\004\167\322\374\145\223\161\146\261\021\225\117\006\214
+END
+CKA_ISSUER MULTILINE_OCTAL
+\060\130\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\042\060\040\006\003\125\004\003\023
+\031\124\162\165\163\164\101\163\151\141\040\124\114\123\040\105
+\103\103\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\066\164\341\115\174\145\023\311\254\203\125\045\240\076
+\122\176\057\120\150\307
+END
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
+#
+# Certificate "TrustAsia TLS RSA Root CA"
+#
+# Issuer: CN=TrustAsia TLS RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:1c:18:d8:cf:e5:53:3f:22:35:46:53:54:24:3c:6c:47:d1:5c:4a:9c
+# Subject: CN=TrustAsia TLS RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:41:57 2024
+# Not Valid After : Sun May 15 05:41:56 2044
+# Fingerprint (SHA-256): 06:C0:8D:7D:AF:D8:76:97:1E:B1:12:4F:E6:7F:84:7E:C0:C7:A1:58:D3:EA:53:CB:E9:40:E2:EA:97:91:F4:C3
+# Fingerprint (SHA1): A5:46:50:C5:62:EA:95:9A:1A:A7:04:6F:17:58:C7:29:53:3D:03:FA
+CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia TLS RSA Root CA"
+CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+CKA_SUBJECT MULTILINE_OCTAL
+\060\130\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\042\060\040\006\003\125\004\003\023
+\031\124\162\165\163\164\101\163\151\141\040\124\114\123\040\122
+\123\101\040\122\157\157\164\040\103\101
+END
+CKA_ID UTF8 "0"
+CKA_ISSUER MULTILINE_OCTAL
+\060\130\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\042\060\040\006\003\125\004\003\023
+\031\124\162\165\163\164\101\163\151\141\040\124\114\123\040\122
+\123\101\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\034\030\330\317\345\123\077\042\065\106\123\124\044\074
+\154\107\321\134\112\234
+END
+CKA_VALUE MULTILINE_OCTAL
+\060\202\005\200\060\202\003\150\240\003\002\001\002\002\024\034
+\030\330\317\345\123\077\042\065\106\123\124\044\074\154\107\321
+\134\112\234\060\015\006\011\052\206\110\206\367\015\001\001\014
+\005\000\060\130\061\013\060\011\006\003\125\004\006\023\002\103
+\116\061\045\060\043\006\003\125\004\012\023\034\124\162\165\163
+\164\101\163\151\141\040\124\145\143\150\156\157\154\157\147\151
+\145\163\054\040\111\156\143\056\061\042\060\040\006\003\125\004
+\003\023\031\124\162\165\163\164\101\163\151\141\040\124\114\123
+\040\122\123\101\040\122\157\157\164\040\103\101\060\036\027\015
+\062\064\060\065\061\065\060\065\064\061\065\067\132\027\015\064
+\064\060\065\061\065\060\065\064\061\065\066\132\060\130\061\013
+\060\011\006\003\125\004\006\023\002\103\116\061\045\060\043\006
+\003\125\004\012\023\034\124\162\165\163\164\101\163\151\141\040
+\124\145\143\150\156\157\154\157\147\151\145\163\054\040\111\156
+\143\056\061\042\060\040\006\003\125\004\003\023\031\124\162\165
+\163\164\101\163\151\141\040\124\114\123\040\122\123\101\040\122
+\157\157\164\040\103\101\060\202\002\042\060\015\006\011\052\206
+\110\206\367\015\001\001\001\005\000\003\202\002\017\000\060\202
+\002\012\002\202\002\001\000\303\026\270\033\152\244\104\163\345
+\326\116\364\271\317\133\013\301\321\232\201\365\143\260\217\103
+\301\273\010\132\032\172\341\007\166\046\037\217\151\126\276\376
+\066\140\320\014\203\315\224\352\347\305\055\134\057\005\026\002
+\236\012\250\057\345\140\046\124\226\370\230\100\035\254\256\235
+\164\147\057\124\373\224\143\230\343\046\112\203\306\225\266\011
+\103\120\315\015\175\336\104\016\140\022\117\133\065\275\277\231
+\070\155\175\154\332\342\150\163\037\371\061\245\031\020\163\005
+\052\303\062\031\205\352\064\252\335\102\036\060\215\077\236\265
+\036\141\325\157\275\000\162\162\255\022\077\252\246\111\163\362
+\206\025\225\014\020\137\121\144\316\377\167\270\311\155\254\345
+\325\230\361\231\056\154\343\311\104\371\265\103\047\010\115\366
+\176\336\104\171\273\262\214\034\332\323\113\154\056\326\303\170
+\267\114\325\244\344\332\334\212\216\016\377\017\303\246\140\046
+\050\317\066\277\372\127\012\354\133\077\351\060\026\173\304\333
+\304\324\144\240\060\373\345\375\035\161\222\335\051\217\101\130
+\336\000\255\072\375\075\174\032\250\261\027\360\116\064\170\130
+\045\326\205\041\353\171\035\320\334\253\317\142\074\260\307\227
+\213\326\320\237\323\376\074\336\305\343\374\072\203\160\204\041
+\035\011\302\241\374\273\050\041\145\123\140\172\220\155\226\070
+\133\377\361\327\172\133\155\323\311\160\111\112\272\035\072\320
+\120\332\062\040\031\344\213\077\353\325\026\046\067\074\373\165
+\236\260\007\160\270\024\140\167\334\366\017\134\122\012\274\135
+\155\215\121\012\332\305\030\310\233\155\334\260\203\263\177\243
+\116\170\114\230\045\253\362\176\056\040\136\202\025\246\326\330
+\217\254\345\315\062\206\310\371\344\332\211\342\073\076\223\305
+\023\152\377\307\367\030\374\147\265\357\030\054\124\253\100\323
+\154\115\355\305\310\267\070\117\144\304\224\114\117\240\315\256
+\245\306\047\045\165\047\155\306\332\234\216\260\005\231\325\050
+\021\305\041\324\270\374\307\226\253\112\144\131\141\153\023\374
+\113\314\222\246\017\244\121\277\016\130\220\331\202\203\071\246
+\137\263\372\156\222\263\315\063\016\205\250\361\051\323\253\131
+\255\326\333\336\324\053\201\002\003\001\000\001\243\102\060\100
+\060\017\006\003\125\035\023\001\001\377\004\005\060\003\001\001
+\377\060\035\006\003\125\035\016\004\026\004\024\270\007\221\171
+\134\006\364\106\375\173\131\312\132\046\221\247\105\053\370\123
+\060\016\006\003\125\035\017\001\001\377\004\004\003\002\001\006
+\060\015\006\011\052\206\110\206\367\015\001\001\014\005\000\003
+\202\002\001\000\041\233\152\005\040\135\030\026\247\022\244\367
+\107\077\315\312\073\256\216\300\202\316\334\110\045\170\027\154
+\340\340\160\304\326\226\331\331\366\277\172\234\023\273\123\225
+\222\377\317\222\340\363\305\047\065\127\250\073\242\031\353\370
+\212\243\004\336\151\213\344\074\074\275\345\257\320\022\033\345
+\211\153\163\233\157\045\002\113\220\202\257\100\302\255\272\232
+\140\044\132\201\115\005\030\243\342\063\171\172\013\037\223\355
+\354\071\112\365\023\004\354\215\164\235\201\012\250\157\350\166
+\376\213\117\053\151\022\206\012\064\066\335\202\233\157\117\340
+\163\335\111\177\050\076\311\073\243\127\004\330\223\331\074\051
+\333\022\020\150\341\304\211\270\006\040\347\033\172\274\261\223
+\141\232\031\100\021\265\152\205\127\001\361\266\132\055\046\163
+\047\226\371\271\077\133\320\210\235\344\240\375\272\116\225\011
+\203\177\167\334\121\005\224\235\040\161\166\261\354\372\211\265
+\234\056\205\361\271\133\044\132\261\011\253\332\312\032\007\315
+\206\337\333\151\333\264\110\030\000\055\274\242\304\211\105\043
+\245\016\341\104\145\076\212\301\152\060\035\342\140\370\072\252
+\207\011\102\271\201\222\334\045\210\230\211\361\174\065\361\142
+\312\356\171\370\031\057\376\137\370\333\207\316\352\250\017\014
+\246\350\013\240\074\340\153\224\230\234\177\255\006\346\034\352
+\102\020\137\342\046\025\074\067\071\367\327\274\356\270\345\357
+\003\356\275\042\373\050\206\064\130\304\132\177\141\242\170\017
+\136\363\312\235\274\033\074\247\310\055\366\247\042\021\312\003
+\330\347\147\012\212\016\313\065\216\064\071\330\310\352\215\237
+\144\340\067\260\154\327\305\217\301\220\141\046\333\044\214\036
+\112\300\210\337\065\220\316\077\002\364\144\141\336\307\310\007
+\225\336\062\030\072\217\242\102\100\044\345\326\063\135\174\256
+\357\261\115\117\324\127\220\065\152\334\256\004\227\111\211\064
+\227\056\060\004\347\230\367\333\013\001\220\301\037\012\077\370
+\302\376\116\372\333\232\163\163\026\274\005\270\171\330\131\257
+\006\347\077\147\070\071\261\174\253\224\340\051\024\246\050\362
+\337\155\342\232\333\124\103\370\107\130\243\135\164\025\234\301
+\253\260\307\064
+END
+CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
+CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+
+# Trust for "TrustAsia TLS RSA Root CA"
+# Issuer: CN=TrustAsia TLS RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Serial Number:1c:18:d8:cf:e5:53:3f:22:35:46:53:54:24:3c:6c:47:d1:5c:4a:9c
+# Subject: CN=TrustAsia TLS RSA Root CA,O="TrustAsia Technologies, Inc.",C=CN
+# Not Valid Before: Wed May 15 05:41:57 2024
+# Not Valid After : Sun May 15 05:41:56 2044
+# Fingerprint (SHA-256): 06:C0:8D:7D:AF:D8:76:97:1E:B1:12:4F:E6:7F:84:7E:C0:C7:A1:58:D3:EA:53:CB:E9:40:E2:EA:97:91:F4:C3
+# Fingerprint (SHA1): A5:46:50:C5:62:EA:95:9A:1A:A7:04:6F:17:58:C7:29:53:3D:03:FA
+CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "TrustAsia TLS RSA Root CA"
+CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+\245\106\120\305\142\352\225\232\032\247\004\157\027\130\307\051
+\123\075\003\372
+END
+CKA_CERT_MD5_HASH MULTILINE_OCTAL
+\073\236\303\206\017\064\074\153\305\106\304\216\035\347\031\022
+END
+CKA_ISSUER MULTILINE_OCTAL
+\060\130\061\013\060\011\006\003\125\004\006\023\002\103\116\061
+\045\060\043\006\003\125\004\012\023\034\124\162\165\163\164\101
+\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
+\054\040\111\156\143\056\061\042\060\040\006\003\125\004\003\023
+\031\124\162\165\163\164\101\163\151\141\040\124\114\123\040\122
+\123\101\040\122\157\157\164\040\103\101
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\034\030\330\317\345\123\077\042\065\106\123\124\044\074
+\154\107\321\134\112\234
+END
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
 #
 # Certificate "D-TRUST EV Root CA 2 2023"
 #
@@ -26138,3 +26578,343 @@ CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
+#
+# Certificate "SwissSign RSA SMIME Root CA 2022 - 1"
+#
+# Issuer: CN=SwissSign RSA SMIME Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Serial Number:46:0e:d4:01:71:90:a0:1a:83:2c:4a:42:10:28:15:d2:61:1b:ad:32
+# Subject: CN=SwissSign RSA SMIME Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Not Valid Before: Wed Jun 08 10:53:13 2022
+# Not Valid After : Sat Jun 08 10:53:13 2047
+# Fingerprint (SHA-256): 9A:12:C3:92:BF:E5:78:91:A0:C5:45:30:9D:4D:9F:D5:67:E4:80:CB:61:3D:63:42:27:8B:19:5C:79:A7:93:1F
+# Fingerprint (SHA1): 14:D7:65:62:74:10:50:47:9F:8B:32:C6:86:8A:18:FA:E1:19:99:B0
+CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "SwissSign RSA SMIME Root CA 2022 - 1"
+CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+CKA_SUBJECT MULTILINE_OCTAL
+\060\123\061\013\060\011\006\003\125\004\006\023\002\103\110\061
+\025\060\023\006\003\125\004\012\023\014\123\167\151\163\163\123
+\151\147\156\040\101\107\061\055\060\053\006\003\125\004\003\023
+\044\123\167\151\163\163\123\151\147\156\040\122\123\101\040\123
+\115\111\115\105\040\122\157\157\164\040\103\101\040\062\060\062
+\062\040\055\040\061
+END
+CKA_ID UTF8 "0"
+CKA_ISSUER MULTILINE_OCTAL
+\060\123\061\013\060\011\006\003\125\004\006\023\002\103\110\061
+\025\060\023\006\003\125\004\012\023\014\123\167\151\163\163\123
+\151\147\156\040\101\107\061\055\060\053\006\003\125\004\003\023
+\044\123\167\151\163\163\123\151\147\156\040\122\123\101\040\123
+\115\111\115\105\040\122\157\157\164\040\103\101\040\062\060\062
+\062\040\055\040\061
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\106\016\324\001\161\220\240\032\203\054\112\102\020\050
+\025\322\141\033\255\062
+END
+CKA_VALUE MULTILINE_OCTAL
+\060\202\005\227\060\202\003\177\240\003\002\001\002\002\024\106
+\016\324\001\161\220\240\032\203\054\112\102\020\050\025\322\141
+\033\255\062\060\015\006\011\052\206\110\206\367\015\001\001\013
+\005\000\060\123\061\013\060\011\006\003\125\004\006\023\002\103
+\110\061\025\060\023\006\003\125\004\012\023\014\123\167\151\163
+\163\123\151\147\156\040\101\107\061\055\060\053\006\003\125\004
+\003\023\044\123\167\151\163\163\123\151\147\156\040\122\123\101
+\040\123\115\111\115\105\040\122\157\157\164\040\103\101\040\062
+\060\062\062\040\055\040\061\060\036\027\015\062\062\060\066\060
+\070\061\060\065\063\061\063\132\027\015\064\067\060\066\060\070
+\061\060\065\063\061\063\132\060\123\061\013\060\011\006\003\125
+\004\006\023\002\103\110\061\025\060\023\006\003\125\004\012\023
+\014\123\167\151\163\163\123\151\147\156\040\101\107\061\055\060
+\053\006\003\125\004\003\023\044\123\167\151\163\163\123\151\147
+\156\040\122\123\101\040\123\115\111\115\105\040\122\157\157\164
+\040\103\101\040\062\060\062\062\040\055\040\061\060\202\002\042
+\060\015\006\011\052\206\110\206\367\015\001\001\001\005\000\003
+\202\002\017\000\060\202\002\012\002\202\002\001\000\324\373\372
+\077\206\242\231\160\011\072\311\326\241\116\001\316\106\265\055
+\044\110\015\105\351\254\311\032\327\062\200\244\346\323\312\326
+\362\051\067\354\232\054\326\201\316\346\033\235\261\017\101\337
+\130\323\137\252\240\171\131\013\214\255\371\305\371\034\373\303
+\351\065\100\164\303\301\014\313\147\373\234\136\014\043\371\320
+\057\372\114\202\105\034\265\365\037\370\031\156\332\223\267\116
+\150\203\176\142\055\035\214\225\037\160\331\316\132\214\041\135
+\250\276\010\251\126\155\331\367\271\220\231\307\073\327\275\236
+\056\311\147\127\056\041\316\360\062\203\373\004\001\371\267\202
+\104\015\262\313\172\011\241\110\076\117\300\013\152\033\033\354
+\317\035\236\177\006\220\254\050\005\013\250\346\124\073\242\247
+\054\071\023\231\250\373\064\353\365\360\275\001\330\210\021\310
+\256\272\031\002\111\002\334\212\054\264\275\141\140\354\235\170
+\134\152\213\057\027\206\136\362\117\223\000\153\135\305\262\251
+\245\030\352\346\155\176\127\005\373\007\374\274\174\241\103\261
+\146\034\250\147\373\253\067\366\252\252\270\172\101\367\071\325
+\214\237\256\162\066\047\344\142\306\103\231\361\241\101\351\377
+\211\237\017\111\257\311\024\006\027\047\144\071\015\245\264\052
+\133\354\074\373\237\305\137\306\073\135\046\031\362\204\051\270
+\225\104\011\122\375\154\060\320\142\217\050\245\201\226\053\224
+\057\046\165\344\011\214\012\337\070\363\176\135\070\216\202\232
+\214\334\236\256\316\022\102\072\042\362\065\215\117\322\032\310
+\111\063\013\372\066\077\377\054\153\152\040\161\114\315\255\020
+\077\151\062\204\216\134\356\123\210\104\342\335\314\003\060\222
+\206\171\220\052\113\165\373\122\142\304\364\157\115\367\333\043
+\241\030\335\105\123\027\232\143\044\110\211\042\315\176\233\001
+\034\200\050\003\217\133\053\372\037\343\174\031\275\346\217\261
+\100\166\240\362\307\067\262\062\224\020\062\315\247\023\227\361
+\073\054\172\064\042\173\164\072\100\342\257\202\370\301\301\170
+\374\137\344\065\177\300\364\114\360\114\262\352\207\003\162\045
+\131\203\052\010\244\121\336\317\356\250\347\350\100\077\210\360
+\066\253\011\116\330\240\250\177\363\341\011\333\271\002\003\001
+\000\001\243\143\060\141\060\017\006\003\125\035\023\001\001\377
+\004\005\060\003\001\001\377\060\016\006\003\125\035\017\001\001
+\377\004\004\003\002\001\006\060\037\006\003\125\035\043\004\030
+\060\026\200\024\314\056\255\211\214\203\343\100\243\045\151\245
+\352\222\175\322\067\072\307\306\060\035\006\003\125\035\016\004
+\026\004\024\314\056\255\211\214\203\343\100\243\045\151\245\352
+\222\175\322\067\072\307\306\060\015\006\011\052\206\110\206\367
+\015\001\001\013\005\000\003\202\002\001\000\000\007\146\026\245
+\355\307\271\277\274\310\221\255\130\355\136\022\005\263\366\106
+\233\173\344\204\020\057\007\261\132\136\062\156\155\000\360\136
+\307\232\142\071\207\115\344\074\363\276\366\272\152\123\154\204
+\155\007\324\141\312\147\146\276\233\076\105\060\235\121\322\327
+\132\102\256\235\222\016\043\253\320\025\224\076\250\232\312\233
+\355\301\302\137\041\243\231\200\374\271\246\141\016\120\200\200
+\047\307\350\175\240\324\224\172\230\003\347\356\102\213\363\122
+\120\134\357\276\260\166\132\034\011\241\277\203\300\036\166\145
+\320\045\101\133\151\221\223\167\371\214\113\360\054\333\233\006
+\000\027\203\372\132\224\264\206\211\357\274\012\013\022\370\006
+\115\322\163\342\221\070\135\271\152\113\117\247\257\332\046\045
+\222\043\032\174\037\047\015\221\204\357\027\162\366\106\353\153
+\073\067\251\324\325\232\143\272\136\171\214\052\265\233\242\064
+\265\314\226\047\371\171\010\074\177\155\340\377\174\334\305\046
+\257\303\241\172\151\026\335\207\065\163\306\153\061\153\126\001
+\055\117\231\331\356\312\341\320\203\336\266\205\347\026\370\321
+\256\063\110\373\032\317\135\137\212\204\141\164\007\157\031\260
+\121\041\262\241\004\261\173\123\217\357\242\105\255\037\230\335
+\371\152\004\154\252\066\305\257\214\352\051\111\112\347\377\127
+\263\306\074\021\111\030\027\346\146\205\341\350\043\362\213\016
+\074\223\304\077\166\116\176\142\242\051\313\026\060\336\054\151
+\257\221\247\342\127\345\063\114\277\330\216\007\242\177\241\065
+\106\151\112\250\040\051\377\353\336\040\225\131\261\256\115\060
+\153\330\232\264\003\037\302\070\344\040\325\016\326\032\060\114
+\150\234\214\340\355\241\017\216\102\135\353\200\052\022\171\246
+\027\027\264\225\052\216\064\221\247\275\321\254\250\032\103\041
+\213\336\357\340\254\106\045\145\012\306\053\241\233\076\042\350
+\030\070\265\046\366\054\312\010\040\336\060\321\255\124\375\334
+\110\025\237\315\247\101\327\067\070\205\060\117\022\210\103\064
+\203\327\052\115\324\000\127\134\345\312\360\335\341\222\050\053
+\030\165\275\107\244\102\023\351\126\132\215\112\136\316\016\165
+\065\200\163\031\017\054\051\213\273\225\270
+END
+CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
+CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+
+# Trust for "SwissSign RSA SMIME Root CA 2022 - 1"
+# Issuer: CN=SwissSign RSA SMIME Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Serial Number:46:0e:d4:01:71:90:a0:1a:83:2c:4a:42:10:28:15:d2:61:1b:ad:32
+# Subject: CN=SwissSign RSA SMIME Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Not Valid Before: Wed Jun 08 10:53:13 2022
+# Not Valid After : Sat Jun 08 10:53:13 2047
+# Fingerprint (SHA-256): 9A:12:C3:92:BF:E5:78:91:A0:C5:45:30:9D:4D:9F:D5:67:E4:80:CB:61:3D:63:42:27:8B:19:5C:79:A7:93:1F
+# Fingerprint (SHA1): 14:D7:65:62:74:10:50:47:9F:8B:32:C6:86:8A:18:FA:E1:19:99:B0
+CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "SwissSign RSA SMIME Root CA 2022 - 1"
+CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+\024\327\145\142\164\020\120\107\237\213\062\306\206\212\030\372
+\341\031\231\260
+END
+CKA_CERT_MD5_HASH MULTILINE_OCTAL
+\232\063\065\330\302\042\023\366\370\153\226\000\022\032\110\141
+END
+CKA_ISSUER MULTILINE_OCTAL
+\060\123\061\013\060\011\006\003\125\004\006\023\002\103\110\061
+\025\060\023\006\003\125\004\012\023\014\123\167\151\163\163\123
+\151\147\156\040\101\107\061\055\060\053\006\003\125\004\003\023
+\044\123\167\151\163\163\123\151\147\156\040\122\123\101\040\123
+\115\111\115\105\040\122\157\157\164\040\103\101\040\062\060\062
+\062\040\055\040\061
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\106\016\324\001\161\220\240\032\203\054\112\102\020\050
+\025\322\141\033\255\062
+END
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
+#
+# Certificate "SwissSign RSA TLS Root CA 2022 - 1"
+#
+# Issuer: CN=SwissSign RSA TLS Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Serial Number:43:fa:0c:5f:4e:1b:80:18:44:ef:d1:b4:4f:35:1f:44:f4:80:ed:cb
+# Subject: CN=SwissSign RSA TLS Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Not Valid Before: Wed Jun 08 11:08:22 2022
+# Not Valid After : Sat Jun 08 11:08:22 2047
+# Fingerprint (SHA-256): 19:31:44:F4:31:E0:FD:DB:74:07:17:D4:DE:92:6A:57:11:33:88:4B:43:60:D3:0E:27:29:13:CB:E6:60:CE:41
+# Fingerprint (SHA1): 81:34:0A:BE:4C:CD:CE:CC:E7:7D:CC:8A:D4:57:E2:45:A0:77:5D:CE
+CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "SwissSign RSA TLS Root CA 2022 - 1"
+CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+CKA_SUBJECT MULTILINE_OCTAL
+\060\121\061\013\060\011\006\003\125\004\006\023\002\103\110\061
+\025\060\023\006\003\125\004\012\023\014\123\167\151\163\163\123
+\151\147\156\040\101\107\061\053\060\051\006\003\125\004\003\023
+\042\123\167\151\163\163\123\151\147\156\040\122\123\101\040\124
+\114\123\040\122\157\157\164\040\103\101\040\062\060\062\062\040
+\055\040\061
+END
+CKA_ID UTF8 "0"
+CKA_ISSUER MULTILINE_OCTAL
+\060\121\061\013\060\011\006\003\125\004\006\023\002\103\110\061
+\025\060\023\006\003\125\004\012\023\014\123\167\151\163\163\123
+\151\147\156\040\101\107\061\053\060\051\006\003\125\004\003\023
+\042\123\167\151\163\163\123\151\147\156\040\122\123\101\040\124
+\114\123\040\122\157\157\164\040\103\101\040\062\060\062\062\040
+\055\040\061
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\103\372\014\137\116\033\200\030\104\357\321\264\117\065
+\037\104\364\200\355\313
+END
+CKA_VALUE MULTILINE_OCTAL
+\060\202\005\223\060\202\003\173\240\003\002\001\002\002\024\103
+\372\014\137\116\033\200\030\104\357\321\264\117\065\037\104\364
+\200\355\313\060\015\006\011\052\206\110\206\367\015\001\001\013
+\005\000\060\121\061\013\060\011\006\003\125\004\006\023\002\103
+\110\061\025\060\023\006\003\125\004\012\023\014\123\167\151\163
+\163\123\151\147\156\040\101\107\061\053\060\051\006\003\125\004
+\003\023\042\123\167\151\163\163\123\151\147\156\040\122\123\101
+\040\124\114\123\040\122\157\157\164\040\103\101\040\062\060\062
+\062\040\055\040\061\060\036\027\015\062\062\060\066\060\070\061
+\061\060\070\062\062\132\027\015\064\067\060\066\060\070\061\061
+\060\070\062\062\132\060\121\061\013\060\011\006\003\125\004\006
+\023\002\103\110\061\025\060\023\006\003\125\004\012\023\014\123
+\167\151\163\163\123\151\147\156\040\101\107\061\053\060\051\006
+\003\125\004\003\023\042\123\167\151\163\163\123\151\147\156\040
+\122\123\101\040\124\114\123\040\122\157\157\164\040\103\101\040
+\062\060\062\062\040\055\040\061\060\202\002\042\060\015\006\011
+\052\206\110\206\367\015\001\001\001\005\000\003\202\002\017\000
+\060\202\002\012\002\202\002\001\000\313\052\150\342\013\303\127
+\274\065\143\274\160\245\073\363\214\074\116\127\226\156\303\116
+\066\244\366\002\312\036\252\256\270\336\250\257\035\166\332\272
+\065\320\221\160\007\337\263\006\362\212\362\056\125\121\173\273
+\054\044\313\177\222\046\200\243\264\224\366\202\241\244\350\372
+\165\035\131\363\007\152\141\144\342\306\214\225\257\243\273\216
+\157\126\317\161\314\136\201\141\015\155\362\253\002\056\244\227
+\345\161\374\212\260\221\040\133\234\164\122\155\256\025\047\131
+\170\362\011\312\145\016\177\313\364\353\347\334\251\114\167\366
+\053\026\004\225\256\234\161\245\077\052\332\101\102\347\074\204
+\020\364\341\075\214\153\342\053\221\107\125\117\270\126\276\105
+\336\042\121\115\116\050\331\137\031\101\006\217\016\115\006\340
+\160\100\043\001\152\344\313\023\233\163\254\115\024\110\222\055
+\376\155\247\370\207\153\171\165\341\276\020\261\252\210\100\131
+\124\327\317\304\320\233\104\263\070\151\144\214\201\321\043\176
+\252\071\074\073\017\237\112\173\202\312\153\157\312\042\076\061
+\320\260\320\052\034\222\212\217\330\031\234\107\344\076\014\271
+\302\315\276\101\014\370\244\107\005\333\301\027\060\070\072\151
+\334\315\303\151\043\375\232\017\002\316\020\152\316\312\370\271
+\051\243\066\211\206\256\013\300\117\143\271\006\131\111\136\016
+\301\151\263\012\363\167\176\056\235\214\263\047\230\322\231\215
+\045\247\037\206\263\246\124\160\070\374\175\135\350\117\203\014
+\321\223\345\022\344\124\332\076\362\255\072\336\076\074\105\360
+\050\017\006\271\341\333\227\173\231\105\236\335\376\225\131\004
+\057\165\077\323\256\211\231\206\254\024\264\250\204\372\310\135
+\073\033\130\223\301\027\224\125\310\013\343\202\171\204\237\363
+\000\204\064\356\334\061\325\217\362\372\117\226\114\006\252\170
+\373\336\144\242\043\315\037\076\305\214\274\067\124\016\273\132
+\162\125\357\310\133\265\162\370\170\337\067\040\114\127\221\163
+\222\163\254\030\167\103\202\040\151\354\351\254\051\106\345\013
+\116\370\067\163\211\226\212\034\155\275\357\276\330\266\364\312
+\300\375\107\360\256\013\130\040\305\310\035\066\256\227\215\120
+\203\046\044\051\367\235\073\017\005\002\003\001\000\001\243\143
+\060\141\060\017\006\003\125\035\023\001\001\377\004\005\060\003
+\001\001\377\060\016\006\003\125\035\017\001\001\377\004\004\003
+\002\001\006\060\037\006\003\125\035\043\004\030\060\026\200\024
+\157\216\142\213\223\103\260\341\100\366\247\303\375\361\017\270
+\017\025\070\245\060\035\006\003\125\035\016\004\026\004\024\157
+\216\142\213\223\103\260\341\100\366\247\303\375\361\017\270\017
+\025\070\245\060\015\006\011\052\206\110\206\367\015\001\001\013
+\005\000\003\202\002\001\000\254\054\051\101\175\372\134\365\032
+\225\030\277\054\251\212\251\044\124\165\365\270\100\253\313\250
+\044\121\053\030\077\143\251\256\230\126\053\005\103\042\243\267
+\327\106\236\300\052\022\075\216\226\226\100\337\014\063\213\153
+\067\221\072\225\273\071\051\155\300\002\154\212\224\013\007\002
+\115\030\076\373\373\173\365\166\075\233\366\136\060\006\130\063
+\036\252\170\325\346\124\004\072\262\202\011\215\316\026\063\131
+\105\050\361\245\243\227\016\103\043\375\013\040\200\220\377\343
+\046\317\270\144\221\345\005\217\023\240\166\015\327\067\014\020
+\210\226\364\076\276\225\275\361\303\175\360\243\303\171\107\013
+\134\222\025\143\355\122\165\212\347\106\151\313\121\125\013\052
+\114\365\362\144\117\251\134\377\147\062\216\125\055\062\202\034
+\200\057\152\221\370\313\274\176\030\242\046\250\056\243\123\050
+\207\355\127\345\145\172\116\000\112\133\116\123\311\142\066\275
+\302\216\133\353\314\156\047\201\030\131\213\104\143\237\325\014
+\145\364\051\145\177\221\054\345\177\176\350\211\317\217\040\313
+\155\007\102\021\121\046\062\212\056\072\107\023\270\215\275\107
+\015\011\360\026\244\355\226\206\056\031\330\276\214\072\350\105
+\056\021\272\256\132\347\271\277\261\314\217\340\240\377\270\263
+\321\205\173\171\146\243\071\265\073\146\330\100\276\317\267\147
+\213\110\311\031\045\125\374\275\215\317\136\332\116\246\362\151
+\316\375\177\114\167\320\301\106\065\230\134\043\233\002\105\103
+\224\132\335\274\107\255\042\376\272\136\057\221\051\051\206\173
+\041\336\156\144\267\313\015\217\067\133\243\010\152\353\364\335
+\002\217\120\003\002\261\270\067\150\226\120\353\270\137\324\050
+\212\245\042\014\212\204\360\131\056\325\067\321\141\345\102\163
+\130\052\201\367\166\333\342\342\115\015\137\366\267\276\005\264
+\256\116\015\336\026\075\003\201\263\046\136\113\270\113\000\317
+\377\214\027\272\154\140\055\047\207\067\044\346\172\140\057\265
+\323\203\004\252\117\103\165\242\301\203\262\047\230\053\261\016
+\200\272\300\205\136\102\271\337\261\140\221\323\353\030\176\160
+\170\256\166\203\276\161\132\320\220\343\312\301\026\045\147\112
+\360\266\173\272\341\234\331
+END
+CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
+CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
+CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
+
+# Trust for "SwissSign RSA TLS Root CA 2022 - 1"
+# Issuer: CN=SwissSign RSA TLS Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Serial Number:43:fa:0c:5f:4e:1b:80:18:44:ef:d1:b4:4f:35:1f:44:f4:80:ed:cb
+# Subject: CN=SwissSign RSA TLS Root CA 2022 - 1,O=SwissSign AG,C=CH
+# Not Valid Before: Wed Jun 08 11:08:22 2022
+# Not Valid After : Sat Jun 08 11:08:22 2047
+# Fingerprint (SHA-256): 19:31:44:F4:31:E0:FD:DB:74:07:17:D4:DE:92:6A:57:11:33:88:4B:43:60:D3:0E:27:29:13:CB:E6:60:CE:41
+# Fingerprint (SHA1): 81:34:0A:BE:4C:CD:CE:CC:E7:7D:CC:8A:D4:57:E2:45:A0:77:5D:CE
+CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "SwissSign RSA TLS Root CA 2022 - 1"
+CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+\201\064\012\276\114\315\316\314\347\175\314\212\324\127\342\105
+\240\167\135\316
+END
+CKA_CERT_MD5_HASH MULTILINE_OCTAL
+\026\056\344\031\166\201\205\272\216\221\130\361\025\357\162\071
+END
+CKA_ISSUER MULTILINE_OCTAL
+\060\121\061\013\060\011\006\003\125\004\006\023\002\103\110\061
+\025\060\023\006\003\125\004\012\023\014\123\167\151\163\163\123
+\151\147\156\040\101\107\061\053\060\051\006\003\125\004\003\023
+\042\123\167\151\163\163\123\151\147\156\040\122\123\101\040\124
+\114\123\040\122\157\157\164\040\103\101\040\062\060\062\062\040
+\055\040\061
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\024\103\372\014\137\116\033\200\030\104\357\321\264\117\065
+\037\104\364\200\355\313
+END
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
+CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
